<h2 id="introduction">Introduction</h2>
<p>     Changes in the extracellular osmolarity can be a fatal event for the bacterial cell. Upon a hypo-osmotic shock, water rushes into the cell across the membrane, leaving the cell with no choice but to equalize the pressure. This equalization occurs either through damage to the cell membrane (resulting in death) or through the regulated flux of water molecules through transmembrane protein channels (Fig 1A). Such proteinaceous pressure release valves have been found across all domains of life, with the first bacterial channel being described in 1987 <span class="citation">(<a href="#ref-martinac1987">1</a>)</span>. Over the past thirty years, several more channels have been discovered, described, and (in many cases) biophysically characterized. <em>E. coli</em>, for example, has seven of these channels (one MscL and six MscS homologs) which have varied conductance, gating mechanisms, and expression levels. While they have been the subject of much experimental and theoretical dissection, much remains a mystery with regard to the roles their abundance and interaction with other cellular processes play in the greater context of physiology <span class="citation">(<a href="#ref-bavi2016">2</a>–<a href="#ref-vandenberg2016">8</a>)</span>.</p>
<p>      Of the seven channels in <em>E. coli</em>, the mechanosensitive channel of large conductance (MscL) is one of the most abundant and the best characterized. This channel has a large conductance (3 nS) and mediates the flux of water molecules across the membrane via a ~3 nm wide pore in the open state <span class="citation">(<a href="#ref-cruickshank1997">9</a>, <a href="#ref-haswell2011">10</a>)</span>. Molecular dynamics simulations indicate that a single open MscL channel permits the flux of <span class="math inline">4 × 10<sup>9</sup></span> water molecules per second, which is an order of magnitude larger than a single aquaporin channel (BNID 100479) <span class="citation">(<a href="#ref-louhivuori2010">11</a>, <a href="#ref-milo2010">12</a>)</span>. This suggests that having only a few channels per cell could be sufficient to relieve even large changes in membrane tension. Electrophysiological experiments have suggested a small number of channels per cell <span class="citation">(<a href="#ref-booth2005">13</a>, <a href="#ref-hase1997">14</a>)</span>, however, more recent approaches using quantitative western blotting, fluorescence microscopy, and proteomics have measured several hundred MscL per cell <span class="citation">(<a href="#ref-bialecka-fornal2012">3</a>, <a href="#ref-schmidt2016">15</a>, <a href="#ref-soufi2015">16</a>)</span>. To further complicate matters, the expression profile of MscL appears to depend on growth phase, available carbon source, and other environmental challenges <span class="citation">(<a href="#ref-bialecka-fornal2012">3</a>, <a href="#ref-soufi2015">16</a>, <a href="#ref-stokes2003a">17</a>)</span>. While there are likely more than just a few channels per cell, why cells seem to need so many and the biological rationale behind their condition-dependent expression both remain a mystery.</p>
<p>      While their biochemical and biophysical characteristics have received much attention, their connection to cell survival is understudied. Drawing such a direct connection between channel copy number and survival requires quantitative <em>in vivo</em> experiments. To our knowledge, the work presented in van den Berg et al. 2016 <span class="citation">(<a href="#ref-vandenberg2016">8</a>)</span> is the first attempt to simultaneously measure channel abundance and survivability for a single species of mechanosensitive channel. While the measurement of channel copy number was performed at the level of single cells using super-resolution microscopy, survivability after a hypo-osmotic shock was assessed in bulk plating assays which rely on serial dilutions of a shocked culture followed by counting the number of resulting colonies after incubation. Such bulk assays have long been the standard for querying cell viability after an osmotic challenge. While they have been highly informative, they reflect only the mean survival rate of the population, obfuscating the variability in survival of the population. The stochastic nature of gene expression results in a noisy distribution of MscL channels rather than a single value, meaning those found in the long tails of the distribution have quite different survival rates than the mean but are lost in the final calculation of survival probability.</p>
<p>       In this work, we present an experimental system to quantitatively probe the interplay between MscL copy number and survival at single-cell resolution, as is seen in Fig. 1B. We generated an <em>E. coli</em> strain in which all seven mechanosensitive channels had been deleted from the chromosome followed by a chromosomal integration of a single gene encoding an MscL-super-folder GFP (sfGFP) fusion protein. To explore copy number regimes beyond those of the wild-type expression level, we modified the Shine-Dalgarno sequence of this integrated construct allowing us to cover nearly three decades of MscL copy number. To probe survivability, we exposed cells to a large hypo-osmotic shock at controlled rates in a flow cell under a microscope, allowing the observation of the single-cell channel copy number and the resulting survivability of single cells. With this large set of single cell measurements, we approach the calculation of survival probability in a manner that is free of binning bias which allows the reasonable extrapolation of survival probability to copy numbers outside of the observed range. In addition, we show that several hundred channels are needed to convey high rates of survival and observe a minimum number of channels needed to permit any degree of survival.</p>
<div class="figure">
<embed src="../figs/fig1.pdf" id="fig:overview" />
<p class="caption">Figure 1: Role of mechanosensitive channels during hypo-osmotic shock. (A) A hypo-osmotic shock results in a large difference in the osmotic strength between the intracellular and extracellular spaces. As a result, water rushes into the cell to equalize this gradient increasing the turgor pressure and tension in the cell membrane. If no mechanosensitive channels are present and membrane tension is high (left panel), the membrane ruptures releasing intracellular content into the environment resulting in cell death . If mechanosensitive channels are present (right panel) and membrane tension is beyond the gating tension, the mechanosensitive channel MscL opens, releasing water and small intracellular molecules into the environment thus relieving pressure and membrane tension. (B) The experimental approach undertaken in this work. The number of mechanosensitive channels tagged with a fluorescent reporter is tuned through modification of the Shine-Dalgarno sequence of the <em>mscL</em> gene. The cells are then subjected to a hypo-osmotic shock and the number of surviving cells are counted, allowing the calculation of a survival probability.</p>
</div>
<h2 id="results">Results</h2>
<h3 id="quantifying-the-single-cell-mscl-copy-number">Quantifying the single-cell MscL copy number</h3>
<p>    The principal goal of this work is to examine the contribution of a single mechanosensitive channel species to cell survival under a hypo-osmotic shock. While this procedure could be performed for any species of channel, we chose MscL as it is the most well characterized and one of the most abundant species in <em>E. coli</em>. To probe the contribution of MscL alone, we generated an <em>E. coli</em> strain in which all seven known mechanosensitive channel genes were deleted from the chromosome followed by the integration of an <em>mscL</em> gene encoding an MscL super-folder GFP (sfGFP) fusion. Chromosomal integration imposes strict control on the gene copy number compared to plasmid borne expression systems, which is important to minimize variation in channel expression across the population and provide conditions more representative of native cell physiology. Fluorescent protein fusions have frequently been used to study MscL and have been shown through electrophysiology to function identically to the native MscL protein, allowing us to confidently draw conclusions about the role this channel plays in wild-type cells from our measurements. <span class="citation">(<a href="#ref-bialecka-fornal2012">3</a>, <a href="#ref-norman2005">18</a>)</span>.</p>
<p>    To modulate the number of MscL channels per cell, we developed a series of mutants which were designed to decrease the expression relative to wild-type. These changes involved direct alterations of the Shine-Dalgarno sequence as well as the inclusion of AT hairpins of varying length directly upstream of the start codon which influences the translation rate and hence the number of MscL proteins produced (Fig. 2A). The six Shine-Dalgarno sequences used in this work were chosen using the RBS binding site strength calculator from the Salis Laboratory at the Pennsylvania State University <span class="citation">(<a href="#ref-espahborujeni2014">19</a>, <a href="#ref-salis2009">20</a>)</span>. While the designed Shine-Dalgarno sequence mutations decreased the expression relative to wild-type as intended, the distribution of expression is remarkably wide spanning an order of magnitude.</p>
<p>    To measure the number of MscL channels per cell, we determined a fluorescence calibration factor to translate arbitrary fluorescence units per cell to protein copy number. While there have been numerous techniques developed over the past decade to directly measure this calibration factor, such as quantifying single-molecule photobleaching constants or measuring the binomial partitioning of fluorescent proteins upon cell division <span class="citation">(<a href="#ref-bialecka-fornal2012">3</a>, <a href="#ref-elowitz2002">21</a>)</span>, we used <em>a priori</em> knowledge of the mean MscL-sfGFP expression level of a particular <em>E. coli</em> strain to estimate the average fluorescence of a single channel. In Bialecka-Fornal et al. 2012 <span class="citation">(<a href="#ref-bialecka-fornal2012">3</a>)</span>, the authors used single-molecule photobleaching and quantitative Western blotting to probe the expression of MscL-sfGFP under a wide range of growth conditions. To compute a calibration factor, we used the strain MLG910 (<em>E. coli</em> K12 MG1655 <span class="math inline"><em>ϕ</em></span>(mscL-sfGFP)) as a &quot;standard candle&quot;, highlighted in yellow in Fig. 2B. This standard candle strain was grown and imaged in identical conditions in which the MscL count was determined. The calibration factor was computed by dividing the mean total cell fluorescence by the known MscL copy number, resulting in a measure of arbitrary fluorescence units per MscL channel. Details regarding this calculation and appropriate propagation of error can be found in the Materials &amp; Methods as well as the supplemental information (<em>Standard Candle Calibration</em>).</p>
<p>    While it is seemingly trivial to use this calibration to determine the total number of channels per cell for wild-type or highly expressing strains, the calculation for the lowest expressing strains is complicated by distorted cell morphology. We observed that as the channel copy number decreases, cellular morphology becomes increasingly aberrant with filamentous, bulging, and branched cells becoming more abundant (Fig. S3A). This morphological defect has been observed when altering the abundance of several species of mechanosensitive channels, suggesting that they play an important role in general architectural stability <span class="citation">(<a href="#ref-bialecka-fornal2012">3</a>, <a href="#ref-bialecka-fornal2015">4</a>)</span>. As these aberrant morphologies can vary widely in size and shape, calculating the number of channels per cell becomes a more nuanced endeavor. For example, taking the total MscL copy number for these cells could skew the final calculation of survival probability as a large but severely distorted cell would be interpreted as having more channels than a smaller, wild-type shaped cell (Fig. S3B). To correct for this pathology, we computed the average expression level per unit area for each cell and multiplied this by the average cellular area of our standard candle strain which is morphologically indistinguishable from wild-type <em>E. coli</em>, allowing for the calculation of an effective channel copy number. The effect of this correction can be seen in Fig. S3C and D, which illustrate that there is no other correlation between cell area and channel expression.</p>
<p>    Our calculation of the effective channel copy number for our suite of Shine-Dalgarno mutants is shown in Fig. 2B. The expression of these strains cover nearly three orders of magnitude with the extremes ranging from approximately four channels per cell to nearly one thousand. While the means of each strain are somewhat distinct, the distributions show a large degree of overlap, making one strain nearly indistinguishable from another. This variance is a quantity that is lost in the context of bulk scale experiments but can be accounted for via single-cell methods.</p>
<div class="figure">
<embed src="../figs/fig2.pdf" id="fig:boxplot" />
<p class="caption">Figure 2: Control of MscL expression and calculation of channel copy number. (A) Schematic view of the expression modifications performed in this work. The beginning portion of the native <em>mscL</em> sequence is shown with the Shine-Dalgarno sequence, spacer region, and start codon shaded in red, green, and blue, respectively. The Shine-Dalgarno sequence was modified through the Salis lab Ribosomal Binding Strength calculator <span class="citation">(<a href="#ref-espahborujeni2014">19</a>, <a href="#ref-salis2009">20</a>)</span>. The wild-type sequence (MLG910) is shown in black with mutations for the other four Shine-Dalgarno mutants highlighted in red. Expression was further modified by the insertion of repetitive <code>AT</code> bases into the spacer region, generating hairpins of varying length which acted as a thermodynamic barrier for translation initiation. (B) Variability in effective channel copy number is computed using the standard candle. The boxes represent the interquartile region of the distribution, the center line displays the median, and the whiskers represent 1.5 times the maximum and minimum of the interquartile region. Individual measurements are denoted as black points. The strain used for calibration of channel copy number (MLG910) is highlighted in yellow.</p>
</div>
<h3 id="performing-a-single-cell-hypo-osmotic-challenge-assay">Performing a single-cell hypo-osmotic challenge assay</h3>
<p>    To measure the channel copy number of a single cell and query its survival after a hypo-osmotic shock, we used a custom-made flow cell in which osmotic shock and growth can be monitored in real time using video microscopy (Fig. 3A). The design and characterization of this device has been described in depth previously and is briefly described in the Materials &amp; Methods <span class="citation">(<a href="#ref-bialecka-fornal2015">4</a>)</span>. Using this device, cells were exposed to a large hypo-osmotic shock by switching between LB Miller medium containing 500mM NaCl and LB media containing no NaCl. All six Shine-Dalgarno modifications shown in Fig. 2B (excluding MLG910) were subjected to a hypo-osmotic shock at controlled rates while under observation. After the application of the osmotic shock, the cells were imaged every sixty seconds for four to six hours. Survivors were defined as cells which underwent at least two divisions post-shock. The brief experimental protocol can be seen in Fig. 3B.</p>
<div class="figure">
<embed src="../figs/fig3.pdf" id="fig:flow_cell" />
<p class="caption">Figure 3: Experimental approach to measuring survival probability. (A) Layout of a home-made flow cell for subjecting cells to osmotic shock. Cells are attached to a polyethylamine functionalized surface of a glass coverslip within the flow chamber by loading a dilute cell suspension through one of the inlets. (B) The typical experimental procedure. Cells are loaded into a flow chamber as shown in (A) and mounted to the glass coverslip surface. Cells are subjected to a hypo-osmotic shock by flowing hypotonic medium into the flow cell. After shock, the cells are monitored for several hours and surviving cells are identified.</p>
</div>
<p>    Due to the extensive overlap in expression between the different Shine-Dalgarno mutants (see Fig. 2B), computing the survival probability by treating each mutant as an individual bin obfuscates the relationship between channel abundance and survival. To more thoroughly examine this relationship, all measurements were pooled together with each cell being treated as an individual experiment. The hypo-osmotic shock applied in these experiments was varied across a range of 0.02 Hz (complete exchange in 50 s) to 2.2 Hz (complete exchange in 0.45 s). Rather than pooling this wide range of shock rates into a single data set, we chose to separate the data into “slow shock” ( &lt; 1.0 Hz) and “fast shock” (<span class="math inline">≥1.0</span> Hz) classes. Other groupings of shock rate were explored and are discussed in the supplemental information (<em>Shock Classification</em>). The cumulative distributions of channel copy number separated by survival are shown in Fig. 4. In these experiments, survival was never observed for a cell containing less than approximately 100 channels per cell, indicated by the red stripe in Fig. 4. This suggests that there is a minimum number of channels needed for survival on the order of 100 per cell. We also observe a slight shift in the surviving fraction of the cells towards higher effective copy number, which matches our intuition that including more mechanosensitive channels increases the survival probability.</p>
<div class="figure">
<embed src="../figs/fig4.pdf" id="fig:survival_dists" />
<p class="caption">Figure 4: Distributions of survival and death as a function of effective channel number. (A) Empirical cumulative distributions of channel copy number separated by survival (green) or death (purple) after a slow (<span class="math inline">&lt;1.0</span> Hz) osmotic shock. (B) The empirical cumulative distribution for a fast (<span class="math inline">≥1.0</span> Hz) osmotic shock. Shaded green and purple regions represent the 95% credible region of the effective channel number calculation for each cell. Shaded red stripe signifies the range of channels in which no survival was observed.</p>
</div>
<h3 id="prediction-of-survival-probability-as-a-function-of-channel-copy-number">Prediction of survival probability as a function of channel copy number</h3>
<p>    There are several ways by which the survival probability can be calculated. The most obvious approach would be to group each individual Shine-Dalgarno mutant as a single bin and compute the average MscL copy number and the survival probability. Binning by strain is the most frequently used approach for such measurements and has provided valuable insight into the qualitative relationship of survival on other physiological factors <span class="citation">(<a href="#ref-bialecka-fornal2015">4</a>, <a href="#ref-vandenberg2016">8</a>)</span>. However the copy number distribution for each Shine-Dalgarno mutant (Fig. 2B) is remarkably wide and overlaps with the other strains. We argue that this coarse-grained binning negates the benefits of performing single-cell measurements as two strains with different means but overlapping quartiles would be treated as distinctly different distributions.</p>
<p>    Another approach would be to pool all data together, irrespective of the Shine-Dalgarno mutation, and bin by a defined range of channels. Depending on the width of the bin, this could allow for finer resolution of the quantitative trend, but the choice of the bin width is arbitrary with the <em>a priori</em> knowledge that is available. Drawing a narrow bin width can easily restrict the number of observed events to small numbers where the statistical precision of the survival probability is lost. On the other hand, drawing wide bins increases the precision of the estimate, but becomes further removed from a true single-cell measurement and represents a population mean, even though it may be a smaller population than binning by the Shine-Dalgarno sequence alone. In both of these approaches, it is difficult to extrapolate the quantitative trend outside of the experimentally observed region of channel copy number. Here, we present a method to estimate the probability of survival for any channel copy number, even those that lie outside of the experimentally queried range.</p>
<p>    To quantify the survival probability while maintaining single-cell resolution, we chose to use a logistic regression model which does not require grouping data into arbitrary bins and treats each cell measurement as an independent experiment. Logistic regression is an inferential method to model the probability of a boolean or categorical event (such as survival or death) given one or several predictor variables and is commonly used in medical statistics to compute survival rates and dose response curves <span class="citation">(<a href="#ref-anderson2003">22</a>, <a href="#ref-mishra2016">23</a>)</span>. The primary assumption of logistic regression is that the log-odds probability of survival <span class="math inline"><em>p</em><sub><em>s</em></sub></span> is linearly dependent on the predictor variable, in our case the log channels per cell <span class="math inline"><em>N</em><sub><em>c</em></sub></span> with a dimensionless intercept <span class="math inline"><em>β</em><sub>0</sub></span> and slope <span class="math inline"><em>β</em><sub>1</sub></span>, <span id="eq:linear_channel_logit" style="display: inline-block; position: relative; width: 100%"><br /><span class="math display">$$
\log{p_s \over 1 - p_s} = \beta_0 + \beta_1 \log N_c.
$$</span><br /><span style="position: absolute; right: 0em; top: 50%; line-height:0; text-align: right">(1)</span></span> Under this assumption of linearity, <span class="math inline"><em>β</em><sub>0</sub></span> is the log-odds probability of survival with no MscL channels. The slope <span class="math inline"><em>β</em><sub>1</sub></span> represents the change in the log-odds probability of survival conveyed by a single channel. As the calculated number of channels in this work spans nearly three orders of magnitude, it is better to perform this regression on <span class="math inline">log<em>N</em><sub><em>c</em></sub></span> as regressing on <span class="math inline"><em>N</em><sub><em>c</em></sub></span> directly would give undue weight for lower channel copy numbers due to the sparse sampling of high-copy number cells. The functional form shown in Eq. <strong>¿eq:linear_channel_logit?</strong> can be derived directly from Bayes’ theorem and is shown in the supplemental information (<em>Logistic Regression</em>). If one knows the values of <span class="math inline"><em>β</em><sub>0</sub></span> and <span class="math inline"><em>β</em><sub>1</sub></span>, the survival probability can be expressed as <span id="eq:prob" style="display: inline-block; position: relative; width: 100%"><br /><span class="math display">$$
p_s = \frac{1}{1 + N_c^{-\beta_1}e^{-\beta_0}}.
$$</span><br /><span style="position: absolute; right: 0em; top: 50%; line-height:0; text-align: right">(2)</span></span> In this analysis, we used Bayesian inferential methods to determine the most likely values of the coefficients and is described in detail in the supplemental information (<em>Logistic Regression</em>).</p>
<p>     The results of the logistic regression are shown in Fig. 5. We see a slight rightward shift the survival probability curve under fast shock relative to the slow shock case, reaffirming the conclusion that survival is also dependent on the rate of osmotic shock <span class="citation">(<a href="#ref-bialecka-fornal2015">4</a>)</span>. This rate dependence has been observed for cells expressing MscL alongside other species of mechanosensitive channels, but not for MscL alone. This suggests that MscL responds differently to different rates of shock, highlighting the need for further study of rate dependence and the coordination between different species of mechanosensitive channels. Fig. 5 also shows that several hundred channels are required to provide appreciable protection from osmotic shock. For a survival probability of 80%, a cell must have approximately 500 to 700 channels per cell for a fast and slow shock, respectively. The results from the logistic regression are showed as continuous colored curves. The individual cell measurements separated by survival and death are shown at the top and bottom of each plot, respectively, and are included to provide a sense of sampling density.</p>
<p>    Over the explored range of MscL copy number, we observed a maximum of 80% survival for any binning method. The remaining 20% survival may be attained when the other species of mechanosensitive channels are expressed alongside MscL. However, it is possible that the flow cell method performed in this work lowers the maximal survival fraction as the cells are exposed to several, albeit minor, mechanical stresses such as loading into the flow cell and chemical adherence to the glass surface. To ensure that the results from logistic regression accurately describe the data, we can compare the survival probabilities to those using the binning methods described earlier (red and black points, Fig. 5). Nearly all binned data fall within error of the prediction (see Materials &amp; Methods for definition of error bar on probability), suggesting that this approach accurately reflects the survival probability and gives license to extrapolate the estimation of survival probability to regions of outside of our experimentally explored copy number regime.</p>
<p>    Thus far, we’ve dictated that for a given rate of osmotic shock (i.e. &quot;fast&quot; or &quot;slow&quot;), the survival probability is dependent only on the number of channels. In Fig. S7, we show the result of including other predictor variables, such as area and shock rate alone. In such cases, including other predictors resulted in pathological curves showing that channel copy number is the most informative out of the available predictor variables.</p>
<div class="figure">
<embed src="../figs/fig5.pdf" id="fig:survival" />
<p class="caption">Figure 5: <strong>Probability of survival as a function of MscL copy number.</strong> (A) Estimated survival probability for survival under slow shock as a function of channel copy number. (B) The estimated survival probability of survival under a fast shock as a function of channel copy number. Solid curves correspond to the most probable survival probability from a one-dimensional logistic regression. Shaded regions represent the 95% credible regions. Points at the top and bottom of plots represent individual cell measurements which survived and perished, respectively. The red and black points correspond to the survival probability estimated via binning by Shine-Dalgarno sequence and binning by groups of 50 channels per cell, respectively. Horizontal error bars represent the standard error of the mean from at least 25 measurements. Vertical error bars represent the certainty of the probability estimate given <span class="math inline"><em>n</em></span> survival events from <span class="math inline"><em>N</em></span> total observations.</p>
</div>
<h2 id="discussion">Discussion</h2>
<p>    One of the most challenging endeavors in the biological sciences is linking the microscopic details of cellular components to the macro-scale physiology of the organism. This formidable task has been met repeatedly in the recent history of biology, especially in the era of DNA sequencing and single molecule biochemistry. For example, the scientific community has been able to connect sickle-cell anemia to a single amino acid substitution in Hemoglobin which promotes precipitation under a change in O<span class="math inline"><em></em><sub>2</sub></span> partial pressure <span class="citation">(<a href="#ref-feeling-taylor2004">24</a>–<a href="#ref-perutz1950">26</a>)</span>. Others have assembled a physical model that quantitatively describes chemosensation in bacteria <span class="citation">(<a href="#ref-berg1977">27</a>)</span> in which the arbiter of sensory adaptation is the repeated methylation of chemoreceptors <span class="citation">(<a href="#ref-colin2017">28</a>–<a href="#ref-sourjik2002">31</a>)</span>. In the past ~50 years alone, numerous biological and physical models of the many facets of the central dogma have been assembled that give us a sense of the interplay between the genome and physiology. For example, the combination of biochemical experimentation and biophysical models have given us a picture of how gene dosage affects furrow positioning in <em>Drosophila</em> <span class="citation">(<a href="#ref-liu2013">32</a>)</span>, how recombination of V(D)J gene segments generates an extraordinarily diverse antibody repertoire <span class="citation">(<a href="#ref-lovely2015">33</a>–<a href="#ref-schatz2011">35</a>)</span>, and how telomere shortening through DNA replication is intrinsically tied to cell senescence <span class="citation">(<a href="#ref-herbig2004">36</a>, <a href="#ref-victorelli2017">37</a>)</span>, to name just a few of many such examples.</p>
<p>     By no means are we ``finished” with any of these topics. Rather, it's quite the opposite in the sense that having a handle on the biophysical knobs that tune the behavior opens the door to a litany of new scientific questions. In the case of mechanosenstaion and osmoregulation, we have only recently been able to determine some of the basic facts that allow us to approach this fascinating biological phenomenon biophysically. The dependence of survival on mechanosensitive channel abundance is a key quantity that is missing from our collection of critical facts. To our knowledge, this work represents the first attempt to quantitatively control the abundance of a single species of mechanosensitive channel and examine the physiological consequences in terms of survival probability at single-cell resolution. Our results reveal two notable quantities. First, out of the several hundred single-cell measurements, we never observed a cell which had less than approximately 100 channels per cell and survived an osmotic shock, irrespective of the shock rate. The second is that between 500 and 700 channels per cell are needed to provide <span class="math inline">≥80%</span> survival, depending on the shock rate.</p>
<p>    Only recently has the relationship between the MscL copy number and the probability of survival been approached experimentally. In van den Berg et al. (2016), the authors examined the contribution of MscL to survival in a genetic background where all other known mechanosensitive channels had been deleted from the chromosome and plasmid-borne expression of an MscL-mEos3.2 fusion was tuned through an IPTG inducible promoter <span class="citation">(<a href="#ref-vandenberg2016">8</a>)</span>. In this work, they measured the single-cell channel abundance through super-resolution microscopy and queried survival through bulk assays. They report a nearly linear relationship between survival and copy number, with approximately 100 channels per cell conveying 100% survival. This number is significantly smaller than our observation of approximately 100 channels as the <em>minimum</em> number needed to convey any observable degree of survival.</p>
<p>     The disagreement between the numbers reported in this work and in van den Berg et al. may partially arise from subtle differences in the experimental approach. The primary practical difference is the rate and magnitude of the osmotic shock. van den Berg et al. applied an approximately 600 mOsm downshock in bulk at an undetermined rate whereas we applied a 1 Osm downshock at controlled rates varying from 0.02 Hz to 2.2 Hz. In their work, survival was measured through plating assays which represent the population average rather than the distribution of survival probability. While this approach provides valuable information regarding the response of a population to an osmotic shock, the high survival rate may stem from a wide distribution of channel copies per cell in the population coupled with bulk-scale measurement of survival. As has been shown in this work, the expression of MscL from a chromosomal integration is noisy with a single strain exhibiting MscL copy numbers spanning an order of magnitude or more. In van den Berg et al., this variance is exacerbated by expression of MscL from an inducible plasmid as fluctuations in the gene copy number from plasmid replication and segregation influence the expression level. Connecting such a wide and complex distribution of copy numbers to single-cell physiology requires the consideration of moments beyond the mean which is a nontrivial task. Rather than trying to make such a connection, we queried survival at single-cell resolution at the expense of a lower experimental throughput.</p>
<p>    Despite these experimental differences, the results of this work and van den Berg et al., are in agreement that MscL must be present at the level of 100 or more channels per cell in wild-type cells to convey appreciable survival. As both of these works were performed in a strain in which the only mechanosensitive channel was MscL, it remains unknown how the presence of the other channel species would alter the number of MscL needed for complete survival. In our experiments, we observed a maximum survival probability of approximately 80% even with close to 1000 MscL channels per cell. It is possible that the combined effort of the six other mechanosensitive channels would make up for some if not all of the remaining 20%. To explore the contribution of another channel to survival, van den Berg et al. also queried the contribution of MscS, another mechanosensitive channel, to survival in the absence of any other species of mechansensitive channel. It was found that over the explored range of MscS channel copy numbers, the maximum survival rate was approximately 50%, suggesting that different mechanosensitive channels have an upper limit to how much protection they can confer. Both van den Berg et al. and our work show that there is still much to be learned with respect to the interplay between the various species of mechanosensitive channel as well as their regulation.</p>
<p>     Recent work has shown that both magnitude and the rate of osmotic down shock are important factors in determining cell survival <span class="citation">(<a href="#ref-bialecka-fornal2015">4</a>)</span>. In this work, we show that this finding holds true for a single species of mechanosensitive channel, even at high levels of expression. One might naïvely expect that this rate-dependent effect would disappear once a certain threshold of channels had been met. Our experiments, however, show that even at nearly 1000 channels per cell the predicted survival curves for a slow (<span class="math inline">&lt;1.0</span> Hz) and fast (<span class="math inline">≥1.0</span> Hz) are shifted relative to each other with the fast shock predicting lower rates of survival. This suggests either we have not reached this threshold in our experiments or there is more to understand about the relationship between abundance, channel species, and the shock rate.</p>
<p>     Some experimental and theoretical treatments suggest that only a few copies of MscL or MscS should be necessary for 100% protection given our knowledge of the conductance and the maximal water flux through the channel in its open state <span class="citation">(<a href="#ref-louhivuori2010">11</a>, <a href="#ref-booth2014">38</a>)</span>. However, recent proteomic studies have revealed average MscL copy numbers to be in the range of several hundred per cell, depending on the condition, as can be seen in Table 1 <span class="citation">(<a href="#ref-schmidt2016">15</a>, <a href="#ref-soufi2015">16</a>, <a href="#ref-li2014">39</a>)</span>. Studies focusing solely on MscL have shown similar counts through quantitative Western blotting and fluorescence microscopy <span class="citation">(<a href="#ref-bialecka-fornal2012">3</a>)</span>. Electrophysiology studies have told another story with copy number estimates ranging between 4 and 100 channels per cell <span class="citation">(<a href="#ref-stokes2003a">17</a>, <a href="#ref-blount1999">40</a>)</span>. These measurements, however, measure the active number of channels. The factors regulating channel activity in these experiments could be due to perturbations during the sample preparation or reflect some unknown mechanism of regulation, such as the presence or absence of interacting cofactors <span class="citation">(<a href="#ref-schumann2010">41</a>)</span>. The work described here, on the other hand, measures the <em>maximum</em> number of channels that could be active and may be able to explain why the channel abundance is higher than estimated by theoretical means. There remains much more to be leared about the regulation of activity in these systems. As the <em>in vivo</em> measurement of protein copy number becomes accessible through novel single-cell and single-molecule methods, we will continue to collect more facts about this fascinating system and hopefully connect the molecular details of mechanosensation with perhaps the most important physiological response -- life or death.</p>
<table>
<caption>Measured cellular copy numbers of MscL. Asterisk (*) Indicates inferred MscL channel copy number from the total number of detected MscL peptides.</caption>
<thead>
<tr class="header">
<th align="left">Reported channels per cell</th>
<th align="left">Method</th>
<th align="center">Reference</th>
</tr>
</thead>
<tbody>
<tr class="odd">
<td align="left">480 ± 103</td>
<td align="left">Western blotting</td>
<td align="center"><span class="citation">(<a href="#ref-bialecka-fornal2012">3</a>)</span></td>
</tr>
<tr class="even">
<td align="left">560*</td>
<td align="left">Ribosomal profiling</td>
<td align="center"><span class="citation">(<a href="#ref-li2014">39</a>)</span></td>
</tr>
<tr class="odd">
<td align="left">331*</td>
<td align="left">Mass spectrometry</td>
<td align="center"><span class="citation">(<a href="#ref-schmidt2016">15</a>)</span></td>
</tr>
<tr class="even">
<td align="left">583*</td>
<td align="left">Mass spectrometry</td>
<td align="center"><span class="citation">(<a href="#ref-soufi2015">16</a>)</span></td>
</tr>
<tr class="odd">
<td align="left">4 - 5</td>
<td align="left">Electrophysiology</td>
<td align="center"><span class="citation">(<a href="#ref-stokes2003a">17</a>)</span></td>
</tr>
<tr class="even">
<td align="left">10 - 100</td>
<td align="left">Electrophysiology</td>
<td align="center"><span class="citation">(<a href="#ref-booth2005">13</a>)</span></td>
</tr>
<tr class="odd">
<td align="left">10 - 15</td>
<td align="left">Electrophysiology</td>
<td align="center"><span class="citation">(<a href="#ref-blount1999">40</a>)</span></td>
</tr>
</tbody>
</table>
<h2 id="materials-methods">Materials &amp; Methods</h2>
<h3 id="bacterial-strains-and-growth-conditions">Bacterial strains and growth conditions</h3>
<p>HJ will summarize here.</p>
<h3 id="flow-cell">Flow cell</h3>
<p>        All experiments were conducted in a home-made flow cell as is shown in Fig. 3A. This flow cell has two inlets which allow media of different osmolarity to be exchanged over the course of the experiment. The imaging region is approximately 10 mm wide and 100 <span class="math inline"><em>μ</em></span>m in depth. All imaging took place within 1 – 2 cm of the outlet to avoid imaging cells within a non-uniform gradient of osmolarity. The interior of the flow cell was functionalized with a 1:400 dilution of polyethylamine prior to addition of cells with the excess washed away with water. A dilute cell suspension in LB Lennox with 500 mM NaCl was loaded into one inlet while the other was connected to a vial of LB medium with no NaCl. This hypotonic medium was clamped during the loading of the cells.</p>
<p>       Once the cells had adhered to the polyethylamine coated surface, the excess cells were washed away with the 500 mM NaCl growth medium followed by a small (~20 <span class="math inline"><em>μ</em></span>L) air bubble. This air bubble forced the cells to lay flat against the imaging surface, improving the time-lapse imaging. Over the observation period, cells not exposed to an osmotic shock were able to grow for 4 – 6 divisions, showing that the flow cell does not directly impede cell growth.</p>
<h3 id="imaging-conditions">Imaging conditions</h3>
<p>       All imaging was performed in a flow cell held at 30°C on a Nikon Ti-Eclipse microscope outfitted with a Perfect Focus system enclosed in a Haison environmental chamber (approximately 1°C regulation efficiency). The microscope was equipped with a 488 nm laser excitation source (CrystaLaser) and a 520/35 laser optimized filter set (Semrock). The images were collected on an Andor Xion +897 EMCCD camera and all microscope and acquisition operations were controlled via the open source <span class="math inline"><em>μ</em></span>Manager microscope control software (27). Once cells were securely mounted onto the surface of the glass coverslip, between 15 and 20 positions containing 5 to 10 cells were marked and the coordinates recorded. At each position, a phase contrast and GFP fluorescence image was acquired for segmentation and subsequent measurement of channel copy number. To perform the osmotic shock, LB media containing no NaCl was pulled into the flow cell through a syringe pump. To monitor the media exchange, both the high salt and no salt LB media were supplemented with a low-affinity version of the calcium-sensitive dye Rhod-2 (250 nM; TEF Labs) which fluoresces when bound to Ca<sup>2+</sup>. The no salt medium was also supplemented with 1<span class="math inline"><em>μ</em></span>M CaCl<sub>2</sub> to make the media mildly fluorescent and the exchange rate was calculated by measuring the fluorescence increase across an illuminated section of one of the positions. These images were collected in real time for the duration of the shock. The difference in measured fluorescence between the pre-shock images and those at the end of the shock set the scale of a 500 mM NaCl down shock. The rate was calculated by fitting a line to the middle region of this trace. Further details regarding this procedure can be found in Bialecka-Fornal, Lee, and Phillips, 2015 <span class="citation">(<a href="#ref-bialecka-fornal2015">4</a>)</span>.</p>
<h3 id="image-processing">Image Processing</h3>
<p>       Images were processed using a combination of automated and manual methods. First, expression of MscL was measured via segmenting individual cells or small clusters of cells in phase contrast and computing the mean pixel value of the fluorescence image for each segmented object. The fluorescence images were passed through several filtering operations which reduced high-frequency noise as well as corrected for uneven illumination of the excitation wavelength.</p>
<p>       Survival or death classification was performed manually using the CellProfiler plugin for ImageJ software (NIH). A survivor was defined as a cell which was able to undergo two division events after the osmotic down shock. Cells which detached from the surface during the post-shock growth phase or those which became indistinguishable from other cells due to clustering were not counted as survival or death and were removed from the dataset completely. A region of the cell was manually marked with 1.0 (survival) or 0.0 (death) by clicking on the image. The <code>xy</code> coordinates of the click as well as the assigned value were saved as an <code>.xml</code> file for that position.</p>
<p>       The connection between the segmented cells and their corresponding manual markers was automated. As the manual markings were made on the first phase contrast image after the osmotic shock, small shifts in the positions of the cell made one-to-one mapping with the segmentation mask non-trivial. The linkages between segmented cell and manual marker were made by computing all pairwise distances between the manual marker and the segmented cell centroid, taking the shortest distance as the true pairing. The linkages were then inspected manually and incorrect mappings were corrected as necessary.</p>
<p>       All relevant statistics about the segmented objects as well as the sample identity, date of acquisition, osmotic shock rate, and camera exposure time were saved as <code>.csv</code> files for each individual experiment. A more in-depth description of the segmentation procedure as well as the relevant code can be accessed as a Jupyter Notebook at (<code>http://rpgroup.caltech.edu/mscl_survival</code>).</p>
<h3 id="calculation-of-effective-channel-copy-number">Calculation of effective channel copy number</h3>
<p>       To compute the MscL channel copy number, we relied on measuring the fluorescence level of a bacterial strain in which the mean MscL channel copy number was known via fluorescence microscopy <span class="citation">(<a href="#ref-bialecka-fornal2012">3</a>)</span>. <em>E. coli</em> strain MLG910, which expresses the MscL-sfGFP fusion protein from the wild-type SD sequence, was grown under identical conditions to those described in Bialecka-Fornal et al. 2015 in M9 minimal medium supplemented with 0.5% glucose to an OD<sub>600nm</sub> of ~0.3. The cells were then diluted ten fold and immobilized on a rigid 2% agarose substrate and placed onto a glass bottom petri dish and imaged in the same conditions as described previously.</p>
<p>     Images were taken of six biological replicates of MLG910 and were processed identically to those in the osmotic shock experiments. A calibration factor between the average cell fluorescence level and mean MscL copy number was then computed. We assumed that all measured fluorescence (after filtering and background subtraction) was derived from the MscL-sfGFP fusion, <span id="eq:ian" style="display: inline-block; position: relative; width: 100%"><br /><span class="math display">⟨<em>I</em><sub>tot</sub>⟩=<em>α</em>⟨<em>N</em>⟩,</span><br /><span style="position: absolute; right: 0em; top: 50%; line-height:0; text-align: right">(3)</span></span> in which <span class="math inline"><em>α</em></span> is the calibration factor and <span class="math inline">⟨<em>N</em>⟩</span> is the mean cellular MscL-sfGFP copy number as reported in Bialecka-Fornal et al. 2012 <span class="citation">(<a href="#ref-bialecka-fornal2012">3</a>)</span>. To correct for errors in segmentation, the intensity was computed as an areal density <span class="math inline">⟨<em>I</em><sub><em>A</em></sub>⟩</span> and was multiplied by the average cell area <span class="math inline">⟨<em>A</em>⟩</span> of the population. The calibration factor was therefore computed as <span id="eq:calibration_factor" style="display: inline-block; position: relative; width: 100%"><br /><span class="math display">$$
\alpha = {\langle I_A \rangle \langle A \rangle \over \langle N \rangle}.
$$</span><br /><span style="position: absolute; right: 0em; top: 50%; line-height:0; text-align: right">(4)</span></span></p>
<p>   We used Bayesian inferential methods to compute this calibration factor taking measurement error and replicate-to-replicate variation into account. The resulting average cell area and calibration factor was used to convert the measured cell intensities from the osmotic shock experiments to cell copy number. The details of this inference are described in depth in the supplemental information (<em>Standard Candle Calibration</em>).</p>
<h3 id="logistic-regression">Logistic regression</h3>
<p>       We used Bayesian inferential methods to find the most probable values of the coefficients <span class="math inline"><em>β</em><sub>0</sub></span> and <span class="math inline"><em>β</em><sub>1</sub></span> and the appropriate credible regions and is described in detail in the supplemental information (<em>Logistic Regression</em>). Briefly, we used Markov chain Monte Carlo (MCMC) to sample from the log posterior distribution and took the most probable value as the mean of the samples for each parameter. The MCMC was performed using the Stan probabilistic programming language <span class="citation">(<a href="#ref-carpenter2017">42</a>)</span> and all models can be found on the GitHub repository (<code>http://github.com/rpgroup-pboc/mscl_survival</code>).</p>
<h3 id="calculation-of-survival-probability-error">Calculation of survival probability error</h3>
<p>    The vertical error bars for the points shown in Fig. 5 represent our uncertainty in the survival probability given our measurement of <span class="math inline"><em>n</em></span> survivors out of a total <span class="math inline"><em>N</em></span> single-cell measurements. The probability distribution of the survival probability <span class="math inline"><em>p</em><sub><em>s</em></sub></span> given these measurements can be written using Bayes' theorem as <span id="eq:probability_bayes" style="display: inline-block; position: relative; width: 100%"><br /><span class="math display">$$
g(p_s\,\vert\, n, N) = {f(n\,\vert\,p_s, N)g(p_s) \over f(n\,\vert\, N)},
$$</span><br /><span style="position: absolute; right: 0em; top: 50%; line-height:0; text-align: right">(5)</span></span> where <span class="math inline"><em>g</em></span> and <span class="math inline"><em>f</em></span> represent probability density functions over parameters and data, respectively. The likelihood <span class="math inline"><em>f</em>(<em>n</em> |<em>p</em><sub><em>s</em></sub>, <em>N</em>)</span> represents the probability of measuring <span class="math inline"><em>n</em></span> survival events, given a total of <span class="math inline"><em>N</em></span> measurements each with a probability of survival <span class="math inline"><em>p</em><sub><em>s</em></sub></span>. This matches the story for the Binomial distribution and can be written as <span id="eq:binomial" style="display: inline-block; position: relative; width: 100%"><br /><span class="math display">$$
f(n\,\vert\,p_s, N) = {N! \over n!(N - n)!}p_s^n(1 - p_s)^{N - n}.
$$</span><br /><span style="position: absolute; right: 0em; top: 50%; line-height:0; text-align: right">(6)</span></span> To maintain maximal ignorance we can assume that any value for <span class="math inline"><em>p</em><sub><em>s</em></sub></span> is valid, such that is in the range [0, 1]. This prior knowledge, represented by <span class="math inline"><em>g</em>(<em>p</em><sub><em>s</em></sub>)</span>, can be written as <span id="eq:uniform_prob" style="display: inline-block; position: relative; width: 100%"><br /><span class="math display">$$
g(p_s) = \begin{cases}1 &amp; 0\leq p_s\leq 1 \\
0 &amp; \text{otherwise} \end{cases}.
$$</span><br /><span style="position: absolute; right: 0em; top: 50%; line-height:0; text-align: right">(7)</span></span> We can also assume maximal ignorance for the total number of survival events we could measure given <span class="math inline"><em>N</em></span> observations, <span class="math inline"><em>f</em>(<em>n</em> | <em>N</em>)</span>. Assuming all observations are equally likely, this can be written as <span id="eq:evidence" style="display: inline-block; position: relative; width: 100%"><br /><span class="math display">$$
f(n\,\vert\, N) = {1 \over N + 1}
$$</span><br /><span style="position: absolute; right: 0em; top: 50%; line-height:0; text-align: right">(8)</span></span> where the addition of one comes from the possibility of observing zero survival events. Combining Eqns. <strong>¿eq:binomial?</strong>, <strong>¿eq:uniform_prob?</strong>, <strong>¿eq:evidence?</strong>, the posterior distribution <span class="math inline"><em>g</em>(<em>p</em><sub><em>s</em></sub> | <em>n</em>, <em>N</em>)</span> is <span id="eq:probability_posterior" style="display: inline-block; position: relative; width: 100%"><br /><span class="math display">$$
g(p_s\,\vert\, n, N) = {(N+1)! \over n!(N - n)!}p_s^{n}(1 - p_s)^{N - n}.
$$</span><br /><span style="position: absolute; right: 0em; top: 50%; line-height:0; text-align: right">(9)</span></span></p>
<p>     The most probable value of <span class="math inline"><em>p</em><sub><em>s</em></sub></span>, where the posterior probability distribution given by Eq. <strong>¿eq:probability_posterior?</strong> is maximized, can be found by computing the point at which derivative of the log posterior with respect to <span class="math inline"><em>p</em><sub><em>s</em></sub></span> goes to zero, <span id="eq:deriv_ps" style="display: inline-block; position: relative; width: 100%"><br /><span class="math display">$$
{d\log g(p_s\,\vert\,n, N) \over d p_s} = {n \over p_s} - {N - n  \over 1 - p_s} = 0.
$$</span><br /><span style="position: absolute; right: 0em; top: 50%; line-height:0; text-align: right">(10)</span></span> Solving Eq. <strong>¿eq:deriv_ps?</strong> for <span class="math inline"><em>p</em><sub><em>s</em></sub></span> gives the most likely value for the probability, <span id="eq:most_prob_prob" style="display: inline-block; position: relative; width: 100%"><br /><span class="math display">$$
p_s^* = {n \over N}.
$$</span><br /><span style="position: absolute; right: 0em; top: 50%; line-height:0; text-align: right">(11)</span></span> So long as <span class="math inline"><em>N</em> &gt; &gt;<em>n</em><em>p</em><sub><em>s</em></sub><sup>*</sup></span>, Eq. <strong>¿eq:probability_posterior?</strong> can be approximated as a Gaussian distribution with a mean <span class="math inline"><em>p</em><sub><em>s</em></sub><sup>*</sup></span> and a variance <span class="math inline"><em>σ</em><sub><em>p</em><sub><em>s</em></sub></sub><sup>2</sup></span>. By definition, the variance of a Gaussian distribution is computed as the negative reciprocal of the second derivative of the log posterior evaluated at <span class="math inline"><em>p</em><sub><em>s</em></sub> = <em>p</em><sub><em>s</em></sub><sup>*</sup></span>, <span id="eq:variance_def" style="display: inline-block; position: relative; width: 100%"><br /><span class="math display">$$
\sigma_{p_s}^2 = - \left({d^2 \log g(p_s\,\vert\, n, N) \over dp_s^2}\Bigg\vert_{p_s=p_s^*}\right)^{-1}.
$$</span><br /><span style="position: absolute; right: 0em; top: 50%; line-height:0; text-align: right">(12)</span></span> Evaluating Eq. <strong>¿eq:variance_def?</strong> yields <span id="eq:prob_variance" style="display: inline-block; position: relative; width: 100%"><br /><span class="math display">$$
\sigma_{p_s}^2 = {n(N-n)\over N^3}.
$$</span><br /><span style="position: absolute; right: 0em; top: 50%; line-height:0; text-align: right">(13)</span></span> Given Eq. <strong>¿eq:most_prob_prob?</strong> and Eq. <strong>¿eq:prob_variance?</strong>, the most-likely survival probability and estimate of the uncertainty can be expressed as <span  style="display: inline-block; position: relative; width: 100%"><br /><span class="math display"><em>p</em><sub><em>s</em></sub> = <em>p</em><sub><em>s</em></sub><sup>*</sup> ± <em>σ</em><sub><em>p</em><sub><em>s</em></sub></sub>.</span><br /><span style="position: absolute; right: 0em; top: 50%; line-height:0; text-align: right">(14)</span></span></p>
<h3 id="data-and-software-availability">Data and software availability</h3>
<p>       All raw image data is freely available and is stored on the CaltechDATA Research Data Repository <span class="citation">(<a href="#ref-chure2018a">43</a>)</span>. The raw Markov chain Monte Carlo samples are stored as <code>.csv</code> files on CaltechDATA <span class="citation">(<a href="#ref-chure2018">44</a>)</span>. All processed experimental data, Python, and Stan code used in this work are freely available through our GitHub repository (<code>http://github.com/rpgroup-pboc/mscl_survival</code>)<span class="citation">(<a href="#ref-chure2018b">45</a>)</span>. The scientific community is invited to fork our repository and open constructive issues.</p>
<h2 id="acknowledgements">Acknowledgements</h2>
<p>       We thank Nathan Belliveau, Maja Bialecka-Fornal, Justin Bois, Soichi Hirokawa, Jaspar Landman, Manuel Razo-Mejia, Muir Morrison, and Shyam Saladi for useful advice and discussion. This work was supported by the National Institutes of Health DP1 OD000217 (Director’s Pioneer Award), R01 GM085286, GM084211-A1 , GM118043-01, and La Fondation Pierre Gilles de Gennes.</p>
<h2 id="references" class="unnumbered">References</h2>
<div id="refs" class="references">
<div id="ref-martinac1987">
<p>1. Martinac B, Buechner M, Delcour AH, Adler J, Kung C. 1987. Pressure-sensitive ion channel in Escherichia coli. Proc Natl Acad Sci U S A 84:2297–301.</p>
</div>
<div id="ref-bavi2016">
<p>2. Bavi N, Cortes DM, Cox CD, Rohde PR, Liu W, Deitmer JW, Bavi O, Strop P, Hill AP, Rees D, Corry B, Perozo E, Martinac B. 2016. The role of MscL amphipathic N terminus indicates a blueprint for bilayer-mediated gating of mechanosensitive channels. Nature Communications 7:11984.</p>
</div>
<div id="ref-bialecka-fornal2012">
<p>3. Bialecka-Fornal M, Lee HJ, DeBerg HA, Gandhi CS, Phillips R. 2012. Single-Cell Census of Mechanosensitive Channels in Living Bacteria. PLoS ONE 7:e33077.</p>
</div>
<div id="ref-bialecka-fornal2015">
<p>4. Bialecka-Fornal M, Lee HJ, Phillips R. 2015. The Rate of Osmotic Downshock Determines the Survival Probability of Bacterial Mechanosensitive Channel Mutants. Journal of Bacteriology 197:231–237.</p>
</div>
<div id="ref-edwards2012">
<p>5. Edwards MD, Black S, Rasmussen T, Rasmussen A, Stokes NR, Stephen TL, Miller S, Booth IR. Jul-Aug 20122012. Characterization of three novel mechanosensitive channel activities in Escherichia coli. Channels (Austin) 6:272–81.</p>
</div>
<div id="ref-naismith2012">
<p>6. Naismith JH, Booth IR. 2012. Bacterial mechanosensitive channels–MscS: Evolution’s solution to creating sensitivity in function. Annu Rev Biophys 41:157–77.</p>
</div>
<div id="ref-ursell2008">
<p>7. Ursell T, Phillips R, Kondev J, Reeves D, Wiggins PA. 2008. The role of lipid bilayer mechanics in mechanosensation, pp. 37–70. <em>In</em> Kamkin, A, Kiseleva, I (eds.), Mechanosensitivity in cells and tissues 1: Mechanosensitive ion channels. Springer-Verlag.</p>
</div>
<div id="ref-vandenberg2016">
<p>8. van den Berg J, Galbiati H, Rasmussen A, Miller S, Poolman B. 2016. On the mobility, membrane location and functionality of mechanosensitive channels in Escherichia coli. Scientific Reports 6.</p>
</div>
<div id="ref-cruickshank1997">
<p>9. Cruickshank CC, Minchin RF, Le Dain AC, Martinac B. 1997. Estimation of the pore size of the large-conductance mechanosensitive ion channel of Escherichia coli. Biophysical Journal 73:1925–1931.</p>
</div>
<div id="ref-haswell2011">
<p>10. Haswell ES, Phillips R, Rees DC. 2011. Mechanosensitive Channels: What Can They Do and How Do They Do It? Structure 19:1356–1369.</p>
</div>
<div id="ref-louhivuori2010">
<p>11. Louhivuori M, Risselada HJ, van der Giessen E, Marrink SJ. 2010. Release of content through mechano-sensitive gates in pressurized liposomes. Proc Natl Acad Sci U S A 107:19856–60.</p>
</div>
<div id="ref-milo2010">
<p>12. Milo R, Jorgensen P, Moran U, Weber G, Springer M. 2010. BioNumbersthe database of key numbers in molecular and cell biology. Nucleic Acids Research 38:D750–D753.</p>
</div>
<div id="ref-booth2005">
<p>13. Booth IR, Edwards MD, Murray E, Miller S. 2005. The role of bacterial ion channels in cell physiology, pp. 291–312. <em>In</em> Kubalsi, A, Martinac, B (eds.), Bacterial Ion Channels and Their Eukaryotic Homologs. American Society for Microbiology, Washington DC.</p>
</div>
<div id="ref-hase1997">
<p>14. Hase CC, Minchin RF, Kloda A, Martinac B. 1997. Cross-linking studies and membrane localization and assembly of radiolabelled large mechanosensitive ion channel (MscL) of Escherichia coli. Biochem Biophys Res Commun 232:777–82.</p>
</div>
<div id="ref-schmidt2016">
<p>15. Schmidt A, Kochanowski K, Vedelaar S, Ahrné E, Volkmer B, Callipo L, Knoops K, Bauer M, Aebersold R, Heinemann M. 2016. The quantitative and condition-dependent Escherichia coli proteome. Nature Biotechnology 34:104–110.</p>
</div>
<div id="ref-soufi2015">
<p>16. Soufi B, Krug K, Harst A, Macek B. 2015. Characterization of the E. coli proteome and its modifications during growth and ethanol stress. Frontiers in Microbiology 6.</p>
</div>
<div id="ref-stokes2003a">
<p>17. Stokes NR, Murray HD, Subramaniam C, Gourse RL, Louis P, Bartlett W, Miller S, Booth IR. 2003. A role for mechanosensitive channels in survival of stationary phase: Regulation of channel expression by RpoS. Proceedings of the National Academy of Sciences 100:15959–15964.</p>
</div>
<div id="ref-norman2005">
<p>18. Norman C, Liu ZW, Rigby P, Raso A, Petrov Y, Martinac B. 2005. Visualisation of the mechanosensitive channel of large conductance in bacteria using confocal microscopy. Eur Biophys J 34:396–402.</p>
</div>
<div id="ref-espahborujeni2014">
<p>19. Espah Borujeni A, Channarasappa AS, Salis HM. 2014. Translation rate is controlled by coupled trade-offs between site accessibility, selective RNA unfolding and sliding at upstream standby sites. Nucleic Acids Research 42:2646–2659.</p>
</div>
<div id="ref-salis2009">
<p>20. Salis HM, Mirsky EA, Voigt CA. 2009. Automated design of synthetic ribosome binding sites to control protein expression. Nature Biotechnology 27:946–950.</p>
</div>
<div id="ref-elowitz2002">
<p>21. Elowitz MB, Levine AJ, Siggia ED, Swain PS. 2002. Stochastic gene expression in a single cell. Science 297:1183–6.</p>
</div>
<div id="ref-anderson2003">
<p>22. Anderson RP, Jin R, Grunkemeier GL. 2003. Understanding logistic regression analysis in clinical reports: An introduction. The Annals of Thoracic Surgery 75:753–757.</p>
</div>
<div id="ref-mishra2016">
<p>23. Mishra V, Skotak M, Schuetz H, Heller A, Haorah J, Chandra N. 2016. Primary blast causes mild, moderate, severe and lethal TBI with increasing blast overpressures: Experimental rat injury model. Scientific Reports 6:26992.</p>
</div>
<div id="ref-feeling-taylor2004">
<p>24. Feeling-Taylor AR, Yau S-T, Petsev DN, Nagel RL, Hirsch RE, Vekilov PG. 2004. Crystallization Mechanisms of Hemoglobin C in the R State. Biophysical Journal 87:2621–2629.</p>
</div>
<div id="ref-finch1973">
<p>25. Finch JT, Perutz MF, Bertles JF, Dobler J. 1973. Structure of Sickled Erythrocytes and of Sickle-Cell Hemoglobin Fibers. Proceedings of the National Academy of Sciences 70:718–722.</p>
</div>
<div id="ref-perutz1950">
<p>26. Perutz MF, Mitchison JM. 1950. State of Hæmoglobin in Sickle-Cell Anæmia. Nature 166:677–679.</p>
</div>
<div id="ref-berg1977">
<p>27. Berg H, Purcell E. 1977. Physics of chemoreception. Biophysical Journal 20:193–219.</p>
</div>
<div id="ref-colin2017">
<p>28. Colin R, Sourjik V. 2017. Emergent properties of bacterial chemotaxis pathway. Current Opinion in Microbiology 39:24–33.</p>
</div>
<div id="ref-krembel2015a">
<p>29. Krembel A, Colin R, Sourjik V. 2015. Importance of Multiple Methylation Sites in Escherichia coli Chemotaxis. PLoS ONE 10.</p>
</div>
<div id="ref-krembel2015">
<p>30. Krembel AK, Neumann S, Sourjik V. 2015. Universal Response-Adaptation Relation in Bacterial Chemotaxis. Journal of Bacteriology 197:307–313.</p>
</div>
<div id="ref-sourjik2002">
<p>31. Sourjik V, Berg HC. 2002. Receptor sensitivity in bacterial chemotaxis. Proceedings of the National Academy of Sciences 99:123–127.</p>
</div>
<div id="ref-liu2013">
<p>32. Liu F, Morrison AH, Gregor T. 2013. Dynamic interpretation of maternal inputs by the Drosophila segmentation gene network. Proceedings of the National Academy of Sciences of the United States of America 110:6724–6729.</p>
</div>
<div id="ref-lovely2015">
<p>33. Lovely GA, Brewster RC, Schatz DG, Baltimore D, Phillips R. 2015. Single-molecule analysis of RAG-mediated V(D)J DNA cleavage. Proceedings of the National Academy of Sciences 112:E1715–E1723.</p>
</div>
<div id="ref-schatz2004">
<p>34. Schatz DG, Baltimore D. 2004. Uncovering the V(D)J recombinase. Cell 116:S103–S108.</p>
</div>
<div id="ref-schatz2011">
<p>35. Schatz DG, Ji Y. 2011. Recombination centres and the orchestration of V(D)J recombination. Nature Reviews Immunology 11:251–263.</p>
</div>
<div id="ref-herbig2004">
<p>36. Herbig U, Jobling WA, Chen BP, Chen DJ, Sedivy JM. 2004. Telomere Shortening Triggers Senescence of Human Cells through a Pathway Involving ATM, p53, and p21CIP1, but Not p16INK4a. Molecular Cell 14:501–513.</p>
</div>
<div id="ref-victorelli2017">
<p>37. Victorelli S, Passos JF. 2017. Telomeres and Cell Senescence - Size Matters Not. EBioMedicine 21:14–20.</p>
</div>
<div id="ref-booth2014">
<p>38. Booth IR. 2014. Bacterial mechanosensitive channels: Progress towards an understanding of their roles in cell physiology. Current Opinion in Microbiology 18:16–22.</p>
</div>
<div id="ref-li2014">
<p>39. Li G-W, Burkhardt D, Gross C, Weissman JS. 2014. Quantifying Absolute Protein Synthesis Rates Reveals Principles Underlying Allocation of Cellular Resources. Cell 157:624–635.</p>
</div>
<div id="ref-blount1999">
<p>40. Blount P, Sukharev SI, Moe PC, Martinac B, Kung C. 1999. Mechanosensitive channels of bacteria. Methods in Enzymology 294:458–482.</p>
</div>
<div id="ref-schumann2010">
<p>41. Schumann U, Edwards MD, Rasmussen T, Bartlett W, van West P, Booth IR. 2010. YbdG in Escherichia coli is a threshold-setting mechanosensitive channel with MscM activity. Proc Natl Acad Sci U S A 107:12664–9.</p>
</div>
<div id="ref-carpenter2017">
<p>42. Carpenter B, Gelman A, Hoffman MD, Lee D, Goodrich B, Betancourt M, Brubaker M, Guo J, Li P, Riddell A. 2017. Stan : A Probabilistic Programming Language. Journal of Statistical Software 76.</p>
</div>
<div id="ref-chure2018a">
<p>43. Chure G, Lee HJ, Phillips R. 2018. Image data for “Connecting the dots between mechanosensitive channel abundance, osmotic shock, and survival at single-cell resolution”.</p>
</div>
<div id="ref-chure2018">
<p>44. Chure G, Lee HJ, Phillips R. 2018. MCMC chains generated in “Connecting the dots between mechanosensitive channel abundance, osmotic shock, and survival at single-cell resolution”.</p>
</div>
<div id="ref-chure2018b">
<p>45. Chure G, Lee HJ, Phillips R. 2018. Github repository for &quot;Connecting the dots between mechanosensitive channel abundance, osmotic shock, and survival at single-cell resolution.</p>
</div>
</div>
