<!DOCTYPE HTML>
<!--
Prologue by HTML5 UP
html5up.net | @ajlkn
Free for personal and commercial use under the CCA 3.0 license (html5up.net/license)
Jekyll integration by Chris Bobbe | chrisbobbe.github.io
-->
<html><head><!-- Robots -->
  <meta name="robots" content="index, follow" /><link rel="canonical" href="http://localhost:4000/" /><!-- Title, description, author --><title>Home | MscL Abundance &amp; Survival</title>
  <meta name="description" content="Home | MscL Abundance &amp;amp; Survival" />
  <meta name="author" content="" />
  
  <!-- Open Graph -->
  <meta property="og:title" content="Home | MscL Abundance &amp; Survival" />
  <meta property="og:type" content="website" />
  <meta property="og:image" content="" />
  <meta property="og:url" content="http://localhost:4000/" />
  <meta property="og:site_name" content="MscL Abundance &amp; Survival" />
  <meta property="og:description" content="Home | MscL Abundance &amp;amp; Survival" />
  
  <!-- Styles -->
  <meta charset="utf-8" />
  <meta name="viewport" content="width=device-width, initial-scale=1" />
  <!--[if lte IE 8]><script src="/assets/js/ie/html5shiv.js" defer></script><![endif]-->
  <link rel="stylesheet" href="/assets/css/main.css" />
  <link href="https://fonts.googleapis.com/css?family=Cabin|Open+Sans" rel="stylesheet"> 

  <!--[if lte IE 8]><link rel="stylesheet" href="/assets/css/ie8.css" /><![endif]-->
  <!--[if lte IE 9]><link rel="stylesheet" href="/assets/css/ie9.css" /><![endif]-->

  <!-- Scripts -->


  <script type="text/x-mathjax-config">
    MathJax.Hub.Config({
    extensions: ["tex2jax.js","color.js"],
    jax: ["input/TeX", "output/HTML-CSS"],
    tex2jax: {
      inlineMath: [ ['$','$'], ["\\(","\\)"] ],
      displayMath: [ ['$$','$$'], ["\\[","\\]"] ],
      processEscapes: true
    },
    "HTML-CSS": {
      preferredFont: "Gyre Pagella",
      availableFonts: ["STIX","TeX", "GyrePagella"]
    }
  });
  </script>
  <script type="text/javascript" src="//cdn.mathjax.org/mathjax/latest/MathJax.js?config=TeX-AMS-MML_HTMLorMML"></script>
  <script src="https://ajax.googleapis.com/ajax/libs/jquery/1.11.3/jquery.min.js" defer></script>
  <script src="/assets/js/jquery.scrolly.min.js" defer></script>
  <script src="/assets/js/jquery.scrollzer.min.js" defer></script>
  <script src="/assets/js/skel.min.js" defer></script>
  <script src="/assets/js/util.js" defer></script>
  <!--[if lte IE 8]><script src="/assets/js/ie/respond.min.js" defer></script><![endif]-->
  <script src="/assets/js/main.js" defer></script>
</head><body><!-- Header -->
<div id="header">

  <div class="top"><!-- Nav -->
<nav id="nav">
  <ul><li><a href="#" id="home" class="active">
            <span class="icon fa-home">Home</span>
          </a></li><li><a href="http://localhost:4000/main.html" id="main-text-link">
            <span class="icon fa-pencil">Main Text</span>
          </a></li><li><a href="http://localhost:4000/supplement.html" id="supplemental-information-link">
            <span class="icon fa-pencil">Supplemental Information</span>
          </a></li><hr><li><a href="#about" id="about-link" class="scrolly"><span class="icon fa-comment">About</span></a></li><li><a href="#abstract" id="abstract-link" class="scrolly"><span class="icon fa-paragraph">Abstract</span></a></li><li><a href="#data-analysis" id="data-analysis-link" class="scrolly"><span class="icon fa-database">Data & Analysis</span></a></li><li><a href="#people" id="people-link" class="scrolly"><span class="icon fa-user">People</span></a></li></ul>
</nav></div>
  <div class="bottom"><!-- Social Icons -->
<ul class="icons"></ul>
</div>
</div>
<!-- Main -->
<div id="main"><!-- Sections --><section id="about" class="shade-one cover dark">
  <div class="container"><img src="/assets/images/composite.png" alt="example cover photo" class="back"><header style="background: rgba(0, 0, 0, 0.5); padding: 1em;">
  <h2 class="alt"> Connecting the dots between mechanosensitive channel abundance, osmotic shock, and survival at single-cell resolution.</h2>
    This website serves as a record for the experimental and statistical inference described in the publication <i>"Connecting the dots between  mechanosensitive channel abundance, osmotic shock, and survival at single-cell resolution"</i> by Griffin Chure <sup>☭</sup>, Heun Jin Lee <sup>☭</sup>, and Rob Phillips. (☭ contributed equally)<br/>
<a href="https://zenodo.org/badge/latestdoi/92714572"><img src="https://zenodo.org/badge/92714572.svg" alt="DOI"></a>
</header>

<footer>
<center>
  <a href="Chure2018a.pdf" class="button scrolly">Main Text (PDF)</a>
  <a href="Chure2018a_supplement.pdf" class="button scrolly">SI (PDF)</a>
  </center> 
</footer>
</div>
</section>
<section id="abstract" class="shade-two">
  <div class="container"><header>
			<h2>Abstract</h2></header><p>Rapid changes in extracellular osmolarity are one of many insults microbial
cells face on a daily basis. To protect against such shocks,
<em>Escherichia coli</em> and other microbes express several types of transmembrane
channels which open and close in response to changes in membrane tension. In
<em>E. coli</em>, the most abundant of these channels is the mechanosensitive
channel of large conductance (MscL). While this channel has been heavily
characterized through structural methods, electrophysiology, and theoretical
modeling, our understanding of its physiological role in preventing cell
death by alleviating high membrane tension remains tenuous. In this work, we
examine the contribution of MscL alone to cell survival after osmotic shock
at single cell resolution using quantitative fluorescence microscopy. We
conduct these experiments in an <em>E. coli</em> strain which is lacking all
mechanosensitive channel genes save for MscL whose expression is tuned across
three orders of magnitude through modifications of the Shine-Dalgarno
sequence. While theoretical models suggest that only a few MscL channels
would be needed to alleviate even large changes in osmotic pressure, we find
that between 500 and 700 channels per cell are needed to convey upwards of
80% survival. This number agrees with the average MscL copy number measured
in wild-type <em>E. coli</em> cells through proteomic studies and quantitative
Western blotting. Furthermore, we observe zero survival events in cells with
less than 100 channels per cell. This work opens new questions concerning the
contribution of other mechanosensitive channels to survival as well as
regulation of their activity.</p>
</div>
</section>
<section id="data-analysis" class="shade-three">
  <div class="container"><header>
			<h2>Data & Analysis</h2></header><p>All single-cell measurements of the osmotic-shock experiments can be downloaded as a <code class="highlighter-rouge">.csv</code> here:</p>
<ul>
  <li><a href="mscl_survival_data.csv"><strong><code class="highlighter-rouge">mscl_survival_data.csv</code></strong></a></li>
</ul>

<p>Raw images and MCMC chains are hosted on the CaltechDATA data repository and can be downloaded through the links below.</p>

<ul>
  <li><a href="https://data.caltech.edu/records/941"><strong>Raw Image Data</strong> (DOI:10.22002/D1.941)</a></li>
  <li><a href="https://data.caltech.edu/records/942"><strong>MCMC Sampling Chains (<code class="highlighter-rouge">.csv</code>)</strong> (DOI:10.22002/D1.942)</a></li>
</ul>

<p>All Python code for processing the raw data sets can be reached through the <code class="highlighter-rouge">master</code> branch of this repository, <a href="http://www.github.com/rpgroup-pboc/mscl_survival.git"><code class="highlighter-rouge">github.com/rpgroup-pboc/mscl_survival.git</code></a>. All MCMC was performed using the <a href="www.mc-stan.org">Stan probabailistic programming language.</a> The <code class="highlighter-rouge">.stan</code> models used for the hierarchical logistic regression can be downloaded below:</p>

<ul>
  <li><a href="code/stan/hierarchical_calbration_factor.stan"><strong><code class="highlighter-rouge">hierarchical_calibration_factor.stan</code></strong></a></li>
  <li><a href="code/stan/hierarchical_logistic_regression.stan"><strong><code class="highlighter-rouge">hierarchical_logistic_regression.stan</code></strong></a></li>
  <li><a href="code/stan/complete_analysis.stan"><strong><code class="highlighter-rouge">complete_analysis.stan</code></strong></a> <strong>|</strong> This model samples both hierarchical models simultaneously.</li>
</ul>

<p>We have also included a Jupyter Notebook which describes the image processing procedure undertaken in this work.</p>

<ul>
  <li><a href="code/notebooks/appendix_A_image_processing.html"><strong>Appendix A:</strong> Image Processing Pipeline</a> <strong>|</strong> <a href="code/notebooks/appendix_A_image_processing.ipynb">download as <code class="highlighter-rouge">.ipynb</code></a></li>
</ul>
</div>
</section>
<section id="people" class="shade-four">
  <div class="container"><header>
			<h2>People</h2></header><p><a href="http://gchure.github.io"><strong>Griffin Chure</strong></a> | Biochemistry &amp; Molecular Biophysics PhD Candidate, California Institute of Technology, Pasadena CA</p>

<p><strong>Heun Jin Lee</strong> | Staff Scientist, California Institute of Technology, Pasadena CA</p>

<p><a href="http://www.rpgroup.caltech.edu/personal_pages/rob/index.html"><strong>Rob Phillips</strong></a> | Phillip and Nancy Morris Professor of Biophysics and Biology, California Institute of Technology, Pasadena CA</p>
</div>
</section>
</div><!-- Footer -->
<div id="footer">
  
  <!-- Copyright -->
  <ul class="copyright">
   
</div></body>
</html>