<!DOCTYPE HTML>
<!--
Prologue by HTML5 UP
html5up.net | @ajlkn
Free for personal and commercial use under the CCA 3.0 license (html5up.net/license)
Jekyll integration by Chris Bobbe | chrisbobbe.github.io
-->
<html><head><!-- Robots -->
  <meta name="robots" content="index, follow" /><link rel="canonical" href="http://localhost:4000/supplement.html" /><!-- Title, description, author --><title>Supplemental Information | MscL Abundance &amp; Survival</title>
  <meta name="description" content="Supplemental Information | MscL Abundance &amp;amp; Survival" />
  <meta name="author" content="" />
  
  <!-- Open Graph -->
  <meta property="og:title" content="Supplemental Information | MscL Abundance &amp; Survival" />
  <meta property="og:type" content="website" />
  <meta property="og:image" content="" />
  <meta property="og:url" content="http://localhost:4000/supplement.html" />
  <meta property="og:site_name" content="MscL Abundance &amp; Survival" />
  <meta property="og:description" content="Supplemental Information | MscL Abundance &amp;amp; Survival" />
  
  <!-- Styles -->
  <meta charset="utf-8" />
  <meta name="viewport" content="width=device-width, initial-scale=1" />
  <!--[if lte IE 8]><script src="/assets/js/ie/html5shiv.js" defer></script><![endif]-->
  <link rel="stylesheet" href="/assets/css/main.css" />
  <link href="https://fonts.googleapis.com/css?family=Cabin|Open+Sans" rel="stylesheet"> 

  <!--[if lte IE 8]><link rel="stylesheet" href="/assets/css/ie8.css" /><![endif]-->
  <!--[if lte IE 9]><link rel="stylesheet" href="/assets/css/ie9.css" /><![endif]-->

  <!-- Scripts -->


  <script type="text/x-mathjax-config">
    MathJax.Hub.Config({
    extensions: ["tex2jax.js","color.js"],
    jax: ["input/TeX", "output/HTML-CSS"],
    tex2jax: {
      inlineMath: [ ['$','$'], ["\\(","\\)"] ],
      displayMath: [ ['$$','$$'], ["\\[","\\]"] ],
      processEscapes: true
    },
    "HTML-CSS": {
      preferredFont: "Gyre Pagella",
      availableFonts: ["STIX","TeX", "GyrePagella"]
    }
  });
  </script>
  <script type="text/javascript" src="//cdn.mathjax.org/mathjax/latest/MathJax.js?config=TeX-AMS-MML_HTMLorMML"></script>
  <script src="https://ajax.googleapis.com/ajax/libs/jquery/1.11.3/jquery.min.js" defer></script>
  <script src="/assets/js/jquery.scrolly.min.js" defer></script>
  <script src="/assets/js/jquery.scrollzer.min.js" defer></script>
  <script src="/assets/js/skel.min.js" defer></script>
  <script src="/assets/js/util.js" defer></script>
  <!--[if lte IE 8]><script src="/assets/js/ie/respond.min.js" defer></script><![endif]-->
  <script src="/assets/js/main.js" defer></script>
</head><body><!-- Header -->
<div id="header">

  <div class="top"><!-- Nav -->
<nav id="nav">
  <ul><li><a href="http://localhost:4000/" id="home-link">
            <span class="icon fa-home">Home</span>
          </a></li><li><a href="http://localhost:4000/main.html" id="main-text-link">
            <span class="icon fa-pencil">Main Text</span>
          </a></li><li><a href="#" id="supplemental-information" class="active">
            <span class="icon fa-pencil">Supplemental Information</span>
          </a></li></ul>
</nav></div>
  <div class="bottom"><!-- Social Icons -->
<ul class="icons"></ul>
</div>
</div>
<!-- Main -->
<div id="main">
	<!-- Page -->
	<article class="shade-two">
	  <div class="container">
			<header>
				<h2>Supplemental Information</h2></header><!-- Header -->
<div id="header">

  <div class="top"><!-- Nav -->
<nav id="nav">
  <ul><li><a href="http://localhost:4000/" id="home-link">
            <span class="icon fa-home">Home</span>
          </a></li><li><a href="http://localhost:4000/main.html" id="main-text-link">
            <span class="icon fa-pencil">Main Text</span>
          </a></li><li><a href="#" id="supplemental-information" class="active">
            <span class="icon fa-pencil">Supplemental Information</span>
          </a></li><hr><li><a href="#candle" id="candle-link" class="scrolly"><span class="icon fa-file">Candle</span></a></li><li><a href="#classification" id="classification-link" class="scrolly"><span class="icon fa-file">Classification</span></a></li><li><a href="#regression" id="regression-link" class="scrolly"><span class="icon fa-file">Regression</span></a></li><li><a href="#strains" id="strains-link" class="scrolly"><span class="icon fa-file">Strains</span></a></li><li><a href="#references" id="references-link" class="scrolly"><span class="icon fa-file">References</span></a></li></ul>
</nav></div>
  <div class="bottom"><!-- Social Icons -->
<ul class="icons"></ul>
</div>
</div>
<!-- Main --><!-- Posts List --><section id="candle" class="shade-two">
  <div class="container"><header>
			<h2><a href="http://localhost:4000/supplement/2018/05/01/candle.html">Candle</a></h2></header><p><p>    To estimate the single-cell MscL abundance via microscopy, we needed to determine a calibration factor that could translate arbitrary fluorescence units to protein copy number. To compute this calibration factor, we relied on <em>a priori</em> knowledge of the mean copy number of MscL-sfGFP for a particular bacterial strain in specific growth conditions. In Bialecka-Fornal et al. 2012 <span class="citation">(1)</span>, the average MscL copy number for a population of cells expressing an MscL-sfGFP fusion (<em>E. coli</em> K-12 MG1655 <span class="math inline">\(\phi\)</span>(<em>mscL-sfGFP</em>)) cells was measured using quantitative Western blotting and single-molecule photobleaching assays. By growing this strain in identical growth and imaging conditions, we can make an approximate measure of this calibration factor. In this section, we derive a statistical model for estimating the most-likely value of this calibration factor and its associated error.</p>
<h3 id="definition-of-a-calibration-factor">Definition of a calibration factor</h3>
<p>    We assume that all detected fluorescence signal from a particular cell is derived from the MscL-sfGFP protein, after background subtraction and correction for autofluorescence. The arbitrary units of fluorescence can be directly related to the protein copy number via a calibration factor <span class="math inline">\(\alpha\)</span>, <span id="eq:ian"><span class="math display">\[
I_\text{tot} = \alpha N_\text{tot},
\qquad(1)\]</span></span> where <span class="math inline">\(I_\text{tot}\)</span> is the total cell fluorescence and <span class="math inline">\(N_\text{tot}\)</span> is the total number of MscL proteins per cell. Bialecka-Fornal et al. report the average cell Mscl copy number for the population rather than the distribution. Knowing only the mean, we can rewrite Eq. 1 as <span id="eq:avg_ian"><span class="math display">\[
\langle I_\text{tot}\rangle = \alpha \langle N_\text{tot} \rangle,
\qquad(2)\]</span></span> assuming that <span class="math inline">\(\alpha\)</span> is a constant value that does not change from cell to cell or fluorophore to fluorophore.</p>
<p>    The experiments presented in this work were performed using non-synchronously growing cultures. As there is a uniform distribution of growth phases in the culture, the cell size distribution is broad the the extremes being small, newborn cells and large cells in the process of division. As described in the main text, the cell size distribution of a population is broadened further by modulating the MscL copy number with low copy numbers resulting in aberrant cell morphology. To speak in the terms of an effective channel copy number, we relate the average areal intensity of the population to the average cell size, <span id="eq:area_conversion"><span class="math display">\[
\langle I_\text{tot} \rangle = \langle I_A \rangle \langle A \rangle = \alpha \langle N_\text{tot} \rangle,
\qquad(3)\]</span></span> where <span class="math inline">\(\langle I_A\rangle\)</span> is the average areal intensity in arbitrary units per pixel of the population and <span class="math inline">\(\langle A \rangle\)</span> is the average area of a segmented cell. As only one focal plane was imaged in these experiments, we could not compute an appropriate volume for each cell given the highly aberrant morphology. We therefore opted to use the projected two-dimensional area of each cell as a proxy for cell size. Given this set of measurements, the calibration factor can be computed as <span id="eq:simple_cal_factor"><span class="math display">\[
\alpha = {\langle I_A \rangle\langle A \rangle \over \langle N_\text{tot} \rangle}.
\qquad(4)\]</span></span></p>
<p>    While it is tempting to use Eq. 4 directly, there are multiple sources of error that are important to propagate through the final calculation. The most obvious error to include is the measurement error reported in Bialecka-Fornal et al. 2012 for the average MscL channel count <span class="citation">(1)</span>. There are also slight variations in expression across biological replicates that arise from a myriad of day-to-day differences. Rather than abstracting all sources of error away into a systematic error budget, we used an inferential model derived from Bayes' theorem that allows for the computation of the probability distribution of <span class="math inline">\(\alpha\)</span>.</p>
<h3 id="estimation-of-alpha-for-a-single-biological-replicate">Estimation of <span class="math inline">\(\alpha\)</span> for a single biological replicate</h3>
<p>    A single data set consists of several hundred single-cell measurements of intensity, area of the segmentation mask, and other morphological quantities. The areal density <span class="math inline">\(I_A\)</span> is computed by dividing the total cell fluorescence by the cell area <span class="math inline">\(A\)</span>. We are interested in computing the probability distributions for the calibration factor <span class="math inline">\(\alpha\)</span>, the average cell area <span class="math inline">\(\langle A \rangle\)</span>, and the mean number of channels per cell <span class="math inline">\(\langle N_\text{tot} \rangle\)</span> for the data set as a whole given only <span class="math inline">\(I_A\)</span> and <span class="math inline">\(A\)</span>. Using Bayes' theorem, the probability distribution for these parameters given a single cell measurement, hereafter called the posterior distribution, can be written as <span id="eq:simple_bayes"><span class="math display">\[
g(\alpha, \langle A \rangle, \langle N_\text{tot} \rangle\,\vert\, A, I_A) = {f(A, I_A\,\vert
\, \alpha,\langle A \rangle, \langle N_\text{tot} \rangle) g(\alpha,\langle A \rangle, \langle N_\text{tot} \rangle) \over f(\alpha, I_A)},
\qquad(5)\]</span></span> where <span class="math inline">\(g\)</span> and <span class="math inline">\(f\)</span> represent probability density functions over parameters and data, respectively. The term <span class="math inline">\(f(A, I_A\,\vert\, \alpha, \langle A \rangle, \langle N_\text{tot} \rangle)\)</span> in the numerator represents the likelihood of observing the areal intensity <span class="math inline">\(I_A\)</span> and area <span class="math inline">\(A\)</span> of a cell for a given values of <span class="math inline">\(\alpha\)</span>, <span class="math inline">\(\langle A \rangle\)</span>, and <span class="math inline">\(\langle N_\text{tot} \rangle\)</span>. The second term in the numerator <span class="math inline">\(g(\alpha,\langle A \rangle, \langle N_\text{tot} \rangle)\)</span> captures all prior knowledge we have regarding the possible values of these parameters knowing nothing about the measured data. The denominator, <span class="math inline">\(f(I_A, A)\)</span> captures the probability of observing the data knowing nothing about the parameter values. This term, in our case, serves simply as a normalization constant and is neglected for the remainder of this section.</p>
<p>    To determine the appropriate functional form for the likelihood and prior, we must make some assumptions regarding the biological processes that generate them. As there are many independent processes that regulate the timing of cell division and cell growth, such as DNA replication and peptidoglycan synthesis, it is reasonable to assume that for a given culture the distribution of cell size would be normally distributed with a mean of <span class="math inline">\(\langle A \rangle\)</span> and a variance <span class="math inline">\(\sigma_{\langle A \rangle}\)</span>. Mathematically, we can write this as <span id="eq:area_likelihood"><span class="math display">\[
f(A\,\vert\,\langle A \rangle, \sigma_{\langle A \rangle}) \propto {1 \over \sigma_{\langle A \rangle}}\exp\left[-{(A - \langle A \rangle)^2 \over 2\sigma_{\langle A \rangle}^2}\right],
\qquad(6)\]</span></span> where the proportionality results from dropping normalization constants for notational simplicity.</p>
<p>     While total cell intensity is intrinsically dependent on the cell area the areal intensity <span class="math inline">\(I_A\)</span> is independent of cell size. The myriad processes leading to the detected fluorescence, such as translation and proper protein folding, are largely independent, allowing us to assume a normal distribution for <span class="math inline">\(I_A\)</span> as well with a mean <span class="math inline">\(\langle I_A \rangle\)</span> and a variance <span class="math inline">\(\sigma_{I_A}^2\)</span>. However, we do not have knowledge of the average areal intensity for the standard candle strain <em>a priori</em>. This can be calculated knowing the calibration factor, total MscL channel copy number, and the average cell area as <span id="eq:avg_rho"><span class="math display">\[
I_A =  {\alpha\langle N_\text{tot} \rangle \over \langle A \rangle}.
\qquad(7)\]</span></span> Using Eq. 7 to calculate the expected areal intensity for the population, we can write the likelihood as a Gaussian distribution, <span id="eq:rho_likelihood"><span class="math display">\[
f(I_A\,\vert\,\alpha,\langle A \rangle,\langle N_\text{tot} \rangle, \sigma_{I_A}) \propto {1 \over \sigma_{I_A}}\exp\left[-{\left(I_A - {\alpha \langle N_\text{tot} \rangle\over \langle A \rangle}\right)^2 \over 2 \sigma_{I_A}^2}\right].
\qquad(8)\]</span></span></p>
<p>    With these two likelihoods in hand, we are tasked with determining the appropriate priors. As we have assumed normal distributions for the likelihoods of <span class="math inline">\(\langle A \rangle\)</span> and <span class="math inline">\(I_A\)</span>, we have included two additional parameters, <span class="math inline">\(\sigma_{\langle A \rangle}\)</span> and <span class="math inline">\(\sigma_{I_A}\)</span>, each requiring their own prior probability distribution. It is common practice to assume maximum ignorance for these variances and use a Jeffreys prior <span class="citation">(2)</span>, <span id="eq:jeffreys"><span class="math display">\[
g(\sigma_{\langle A \rangle}, \sigma_{I_A}) = {1 \over \sigma_{\langle A \rangle}\sigma_{I_A}}.
\qquad(9)\]</span></span></p>
<p>    The next obvious prior to consider is for the average channel copy number <span class="math inline">\(\langle N_\text{tot} \rangle\)</span>, which comes from Bialecka-Fornal et al. 2012. In this work, they report a mean <span class="math inline">\(\mu_N\)</span> and variance <span class="math inline">\(\sigma_N^2\)</span>, allowing us to assume a normal distribution for the prior, <span id="eq:informative_prior"><span class="math display">\[
g(\langle N_\text{tot}\rangle\,\vert\, \mu_N,\sigma_N) \propto {1 \over \sigma_N}\exp\left[-{(\langle N_\text{tot} \rangle - \mu_N)^2 \over 2 \sigma_N^2}\right].
\qquad(10)\]</span></span> For <span class="math inline">\(\alpha\)</span> and <span class="math inline">\(\langle A \rangle\)</span>, we have some knowledge of what these parameters can and cannot be. For example, we know that neither of these parameters can be negative. As we have been careful to not overexpose the microscopy images, we can say that the maximum value of <span class="math inline">\(\alpha\)</span> would be the bit-depth of our camera. Similarly, it is impossible to segment a single cell with an area larger than our camera's field of view (although there are biological limitations to size below this extreme). To remain maximally uninformative, we can assume that the parameter values are uniformly distributed between these bounds, allowing us to state <span id="eq:alpha_uniform_prior"><span class="math display">\[
g(\alpha) = \begin{cases} {1 \over \alpha_\text{max} - \alpha_\text{min}} &amp; \alpha_\text{min} \leq \alpha \leq \alpha_\text{max} \\
0 &amp; \text{otherwise}
\end{cases},
\qquad(11)\]</span></span> for <span class="math inline">\(\alpha\)</span> and <span id="eq:area_uniform_prior"><span class="math display">\[
g(\langle A \rangle) = \begin{cases} {1 \over \langle A \rangle_\text{max} - \langle A \rangle_\text{min}} &amp; \langle A \rangle_\text{min} \leq \langle A \rangle \leq \langle A \rangle_\text{max}\\
0 &amp; \text{otherwise}
\end{cases}
\qquad(12)\]</span></span> for <span class="math inline">\(\langle A \rangle\)</span>.</p>
<p>Piecing Eq. 6 through Eq. 12 together generates a complete posterior probability distribution for the parameters given a single cell measurement. This can be generalized to a set of <span class="math inline">\(k\)</span> single cell measurements as <span id="eq:single_rep_post"><span class="math display">\[
\begin{aligned}
g(\alpha,\langle A \rangle, \langle N_\text{tot} \rangle, \sigma_{I_A}, \sigma_{\langle A \rangle}\,\vert\, [I_A, A], \mu_N, \sigma_N) &amp;\propto {1 \over (\alpha_\text{max} - \alpha_\text{min})(\langle A \rangle_\text{max} - \langle A \rangle_\text{min})}{1 \over (\sigma_{I_A}\sigma_{\langle A \rangle})^{k+1}}\,\times\\
&amp;{1 \over \sigma_N}\exp\left[- {(\langle N_\text{tot}\rangle - \mu_N)^2 \over 2\sigma_N^2}\right]
\prod\limits_i^k\exp\left[-{(A^{(i)} - \langle A \rangle)^2 \over 2\sigma_{\langle A \rangle}^2} - {\left(I_A^{(i)} - {\alpha \langle N_\text{tot}\rangle \over \langle A \rangle}\right)^2 \over 2\sigma_{I_A}^2}\right] \end{aligned},
\qquad(13)\]</span></span> where <span class="math inline">\([I_A, A]\)</span> represents the set of <span class="math inline">\(k\)</span> single-cell measurements.</p>
<p>    As small variations in the day-to-day details of cell growth and sample preparation can alter the final channel count of the standard candle strain, it is imperative to perform more than a single biological replicate. However, properly propagating the error across replicates is non trivial. One option would be to pool together all measurements of <span class="math inline">\(n\)</span> biological replicates and evaluate the posterior given in Eq. 13. However, this by definition assumes that there is no difference between replicates. Another option would be to perform this analysis on each biological replicate individually and then compute a mean and standard deviation of the resulting most-likely parameter estimates for <span class="math inline">\(\alpha\)</span> and <span class="math inline">\(\langle A \rangle\)</span>. While this is a better approach than simply pooling all data together, it suffers a bias from giving each replicate equal weight, skewing the estimate of the most-likely parameter value if one replicate is markedly brighter or dimmer than the others. Given this type of data and a limited number of biological replicates (<span class="math inline">\(n = 6\)</span> in this work), we chose to extend the Bayesian analysis presented in this section to model the posterior probability distribution for <span class="math inline">\(\alpha\)</span> and <span class="math inline">\(\langle A \rangle\)</span> as a hierarchical process in which <span class="math inline">\(\alpha\)</span> and <span class="math inline">\(\langle A \rangle\)</span> for each replicate is drawn from the same distribution.</p>
<h3 id="a-hierarchical-model-for-estimating-alpha">A hierarchical model for estimating <span class="math inline">\(\alpha\)</span></h3>
<p>    In the previous section, we assumed maximally uninformative priors for the most-likely values of <span class="math inline">\(\alpha\)</span> and <span class="math inline">\(\langle A \rangle\)</span>. While this is a fair approach to take, we are not completely ignorant with regard to how these values are distributed across biological replicates. A major assumption of our model is that the most-likely value of <span class="math inline">\(\alpha\)</span> and <span class="math inline">\(\langle A \rangle\)</span> for each biological replicate are comparable, so long as the experimental error between them is minimized. In other words, we are assuming that the most-likely value for each parameter for each replicate is drawn from the same distribution. While each replicate may have a unique value, they are all related to one another. Unfortunately, proper sampling of this distribution requires an extensive amount of experimental work, making inferential approaches more attractive.</p>
<p>This approach, often called a multi-level or hierarchical model, is schematized in Fig. 1. Here, we use an informative prior for <span class="math inline">\(\alpha\)</span> and <span class="math inline">\(\langle A \rangle\)</span> for each biological replicate. This informative prior probability distribution can be described by summary statistics, often called hyper-parameters, which are then treated as the &quot;true&quot; value and are used to calculate the channel copy number. As this approach allows us to get a picture of the probability distribution of the hyper-parameters, we are able to report a point estimate for the most-likely value along with an error estimate that captures all known sources of variation.</p>
<div class="figure">
<embed src="assets/hierarchical_model.png" id="fig:hierarchical_model" />
<p class="caption">Figure 1: Schematic of hierarchical model structure. The hyper-parameter probability distributions (top panel) are used as an informative prior for the most-likely parameter values for each biological replicate (middle panel). The single-cell measurements of cell area and areal intensity (bottom panel) are used as data in the evaluation of the likelihood.</p>
</div>
<p>    The choice for the functional form for the informative prior is often not obvious and can require other experimental approaches or back-of-the-envelope estimates to approximate. Each experiment in this work was carefully constructed to minimize the day-to-day variation. This involved adhering to well-controlled growth temperatures and media composition, harvesting of cells at comparable optical densities, and ensuring identical imaging parameters. As the experimental variation is minimized, we can use our knowledge of the underlying biological processes to guess at the approximate functional form. For similar reasons presented in the previous section, cell size is controlled by a myriad of independent processes. As each replicate is independent of another, it is reasonable to assume a normal distribution for the average cell area for each replicate. This normal distribution is described by a mean <span class="math inline">\(\tilde{\langle A \rangle}\)</span> and variance <span class="math inline">\(\tilde{\sigma}_{\langle A \rangle}\)</span>. Therefore, the prior for <span class="math inline">\(\langle A \rangle\)</span> for <span class="math inline">\(n\)</span> biological replicates can be written as <span id="eq:hyperprior_area"><span class="math display">\[
g(\langle A \rangle\, \vert\, \tilde{\langle A \rangle}, \tilde{\sigma}_{\langle A \rangle}) \propto {1 \over \tilde{\sigma}_{\langle A \rangle}^n}\prod\limits_{j=1}^{n}\exp\left[-{(\langle A \rangle_j - \tilde{\langle A \rangle})^2 \over 2 \tilde{\sigma}_{\langle A \rangle}^2}\right].
\qquad(14)\]</span></span> In a similar manner, we can assume that the calibration factor for each replicate is normally distributed with a mean <span class="math inline">\(\tilde{\alpha}\)</span> and variance <span class="math inline">\(\tilde{\sigma}_\alpha\)</span>, <span id="eq:hyperprior_alpha"><span class="math display">\[
g(\alpha\,\vert\,\tilde{\alpha}, \tilde{\sigma}_\alpha) \propto {1 \over \tilde{\sigma}_\alpha^n}\prod\limits_{j=1}^n \exp\left[-{(\alpha_j - \tilde{\alpha})^2 \over 2\tilde{\sigma}_\alpha^2}\right].
\qquad(15)\]</span></span></p>
<p>    With the inclusion of two more normal distributions, we have introduced four new parameters, each of which needing their own prior. However, our knowledge of the reasonable values for the hyper-parameters has not changed from those described for a single replicate. We can therefore use the same maximally uninformative Jeffreys priors given in Eq. 9 for the variances and the uniform distributions given in Eq. 11 and Eq. 12 for the means. Stitching all of this work together generates the full posterior probability distribution for the best-estimate of <span class="math inline">\(\tilde{\alpha}\)</span> and <span class="math inline">\(\tilde{\langle A \rangle}\)</span> shown in Eq. 2 given <span class="math inline">\(n\)</span> replicates of <span class="math inline">\(k\)</span> single cell measurements, <span id="eq:cal_factor_posterior"><span class="math display">\[
\begin{aligned}
g(\tilde{\alpha}, \tilde{\sigma}_\alpha, \tilde{\langle A \rangle}, \tilde{\sigma}_{\langle A \rangle}, &amp;\{\langle N_\text{tot} \rangle, \langle A \rangle, \alpha, \sigma_{I_A}\}\,\vert\, [I_A, A], \mu_N, \sigma_N) \propto\\
&amp;{1 \over (\tilde{\alpha}_\text{max} - \tilde{\alpha}_\text{min})(\tilde{\langle A \rangle}_\text{max} - \tilde{\langle A \rangle}_\text{min})\sigma_N^n(\tilde{\sigma}_\alpha\tilde{\sigma}_{\langle A \rangle})^{n + 1}}\,\times\,\\
&amp;\prod\limits_{j=1}^n\exp\left[-{(\langle N \rangle_j^{(i)} - \mu_N)^2 \over 2\sigma_N^2} - {(\alpha_j - \tilde{\alpha})^2 \over 2\tilde{\sigma}_\alpha^2} - {(\langle A \rangle_j - \tilde{\langle A \rangle})^2 \over 2\tilde{\sigma}_{\langle A \rangle}^2}\right]\,\times\,\\ 
&amp;{1 \over (\sigma_{{I_A}_j}\sigma_{\langle A \rangle_j})^{k_j + 1}}\prod\limits_{i=1}^{k_j}\exp\left[-{(A_j^{(i)} - \langle A \rangle_j)^2 \over 2\sigma^{(i)2}_{\langle A \rangle_j}} - {\left({I_A}_{j}^{(i)} - {\alpha_j \langle N_\text{tot}\rangle_j \over \langle A \rangle_j}\right)\over 2\sigma_{{I_A}_j}^{(i)2}}\right]
\end{aligned},
\qquad(16)\]</span></span> where the braces <span class="math inline">\(\{\dots\}\)</span> represent the set of parameters for biological replicates and the brackets <span class="math inline">\([\dots]\)</span> correspond to the set of single-cell measurements for a given replicate.</p>
<p>    While Eq. 16 is not analytically solvable, it can be easily sampled using Markov chain Monte Carlo (MCMC). The results of the MCMC sampling for <span class="math inline">\(\tilde{\alpha}\)</span> and <span class="math inline">\(\tilde{\langle A \rangle}\)</span> can be seen in Fig. 2. From this approach, we found the most-likely parameter values of <span class="math inline">\(3300^{+700}_{-700}\)</span> a.u. per MscL channel and <span class="math inline">\(5.4^{+0.4}_{-0.5}\)</span> <span class="math inline">\(\mu\)</span>m<span class="math inline">\(^2\)</span> for <span class="math inline">\(\tilde{\alpha}\)</span> and <span class="math inline">\(\tilde{\langle A \rangle}\)</span>, respectively. Here, we've reported the median value of the posterior distribution for each parameter with the upper and lower bound of the 95% credible region as superscript and subscript, respectively. These values and associated errors were used in the calculation of channel copy number.</p>
<div class="figure">
<embed src="assets/figS3.png" id="fig:posterior_samples" />
<p class="caption">Figure 2: Posterior distributions for hyper-parameters and replicate parameters. (A) The posterior probability distribution for <span class="math inline">\(\tilde{\alpha}\)</span> and <span class="math inline">\(\tilde{\langle A \rangle}\)</span>. Probability increases from light to dark red. The replicate parameter (blue) and hyper-parameter (red) marginalized posterior probability distributions for <span class="math inline">\(\alpha\)</span> (B) and <span class="math inline">\(\langle A \rangle\)</span> (C).</p>
</div>
<h3 id="effect-of-correction">Effect of correction</h3>
<p>     The posterior distributions for <span class="math inline">\(\alpha\)</span> and <span class="math inline">\(\langle A \rangle\)</span> shown in Fig. 2 were used directly to compute the most-likely channel copy number for each measurement of the Shine-Dalgarno mutant strains, as is described in the coming section (<em>Logistic Regression</em>). The importance of this correction can be seen in Fig. 3. Cells with low abundance of MscL channels exhibit notable morphological defects, as illustrated in Fig. 3A. While these would all be considered single cells, the two-dimensional area of each may be comparable to two or three wild-type cells. For all of the Shine-Dalgarno mutants, the distribution of projected cell area has a long tail, with the extremes reaching 35 µm<span class="math inline">\(^2\)</span> per cell (Fig. 3B). Calcuating the total number of channels per cell does nothing to decouple this correlation between cell area and measured cell intensity. Fig. 3C shows the correlation between cell area and the total number of channels without normalizing to an average cell size <span class="math inline">\(\langle A \rangle\)</span> differentiated by their survival after an osmotic down-shock. This correlation is removed by calculating an effective channel copy number shown in Fig. 3D.</p>
<div class="figure">
<embed src="assets/figS2.png" id="fig:area_correction" />
<p class="caption">Figure 3: <strong>Influence of area correction for Shine-Dalgarno mutants.</strong> (A) Representative images of aberrant cell morphologies found in low-expressing Shine-Dalgarno mutants. (B) Empirical cumulative distribution of two-dimensional projected cell area for the standard candle strain MLG910 (gray line) and for all Shine-Dalgarno mutants (red line). (C) The correlation between channel copy number and cell area without the area correction. (D) The correlation between effective channel copy number and cell area with the area correction applied.</p>
</div>
</p></div>
</section>
<section id="classification" class="shade-three">
  <div class="container"><header>
			<h2><a href="http://localhost:4000/supplement/2018/05/02/classification.html">Classification</a></h2></header><p><p>    Its been previously shown that the rate of hypo-osmotic shock dictates the survival probability <span class="citation">(3)</span>. To investigate how a single channel contributes to survival, we queried survival at several shock rates with varying MscL copy number. In the main text of this work, we separated our experiments into arbitrary bins of &quot;fast&quot; (<span class="math inline">\(\geq\)</span> 1.0 Hz) and &quot;slow&quot; (<span class="math inline">\(&lt;\)</span> 1.0 Hz) shock rates. In this section, we discuss our rationale for coarse graining our data into these two groupings.</p>
<p>    As is discussed in the main text and in the supplemental section <em>Logistic Regression</em>, we used a bin-free method of estimating the survival probability given the MscL channel copy number as a predictor variable. While this method requires no binning of the data, it requires a data set that sufficiently covers the physiological range of channel copy number to accurately allow prediction of survivability. Fig. 4 shows the results of the logistic regression treating each shock rate as an individual data set. The most striking feature of the plots shown in Fig. 4 is the inconsistent behavior of the predicted survivability from shock rate to shock rate. The appearance of bottle necks in the credible regions for some shock rates (0.2Hz, 0.5Hz, 2.00Hz, and 2.20 Hz) appear due to a high density of measurements within a narrow range of the channel copy number at the narrowest point in the bottle neck. While this results in a seemingly accurate prediction of the survival probability at that point, the lack of data in other copy number regimes severely limits our extrapolation outside of the copy number range of that data set. Other shock rates (0.018 Hz, 0.04 Hz, and 1.00 Hz) demonstrate completely pathological survival probability curves due to either complete survival or complete death of the population.</p>
<p>    Ideally, we would like to have a wide range of MscL channel copy numbers at each shock rate shown in Fig. 4. However, the low-throughput nature of these single-cell measurements prohibits completion of this within a reasonable time frame. It is also unlikely that thoroughly dissecting the shock rate dependence will change the overall finding from our work that several hundred MscL channels are needed to convey survival under hypo-osmotic stress.</p>
<div class="figure">
<img src="assets/figS5_all_indiv_shock_regression.png" alt="Figure 4: Binning by individual shock rates. Survival probability estimates from logistic regression (red lines) and the computed survival probability for all SD mutants subjected to that shock rate (blue points). Black points at top and bottom of each plot correspond to single cell measurements of survival (top) and death (bottom). Red shaded regions signify the 95% credible region of the logistic regression. Horizontal error bars of blue points are the standard error of the mean channel copy number. Vertical error bars of blue points correspond to the uncertainty in survival probability by observing n survival events from N single-cell measurements." id="fig:indiv_shock_group" />
<p class="caption">Figure 4: Binning by individual shock rates. Survival probability estimates from logistic regression (red lines) and the computed survival probability for all SD mutants subjected to that shock rate (blue points). Black points at top and bottom of each plot correspond to single cell measurements of survival (top) and death (bottom). Red shaded regions signify the 95% credible region of the logistic regression. Horizontal error bars of blue points are the standard error of the mean channel copy number. Vertical error bars of blue points correspond to the uncertainty in survival probability by observing <span class="math inline">\(n\)</span> survival events from <span class="math inline">\(N\)</span> single-cell measurements.</p>
</div>
<p>     Given the data shown in Fig. 4, we can try to combine the data sets into several bins. Fig. 5 shows the data presented in Fig. 4 separated into &quot;slow&quot; (<span class="math inline">\(&lt;\)</span> 0.5 Hz, A), &quot;intermediate&quot; (0.5 - 1.0 Hz, B), and &quot;fast&quot; (<span class="math inline">\(&gt;\)</span> 1.0 Hz, C) shock groups. Using these groupings, the full range of MscL channel copy numbers are covered for each case, with the intermediate shock rate sparsely sampling copy numbers greater than 200 channels per cell. In all three of these cases, the same qualitative story is told -- several hundred channels per cell are necessary for an appreciable level of survival when subjected to an osmotic shock. This argument is strengthened when examining the predicted survival probability by considering all shock rates as a single group, shown in Fig. 5D. This treatment tells nearly the same quantitative and qualitative story as the three rate grouping shown in this section and the two rate grouping presented in the main text. While there does appear to be a dependence on the shock rate for survival when only MscL is expressed, the effect is relatively weak with overlapping credible regions for the logistic regression across the all curves. To account for the sparse sampling of high copy numbers observed in the intermediate shock group, we split this set and partitioned the measurements into either the &quot;slow&quot; (<span class="math inline">\(&lt;\)</span> 1.0 Hz) or &quot;fast&quot; (<span class="math inline">\(\geq\)</span> 1.0 Hz) groups presented in the main text of this work.</p>
<div class="figure">
<img src="assets/figS4_alternative_binning.png" alt="Figure 5: Coarse graining shock rates into different groups. Estimated survival probability curve for slow (A), intermediate (B), and fast (C) shock rates. (D) Estimated survival probability curve from pooling all data together, ignoring varying shock rates. Red shaded regions correspond to the 95% credible region of the survival probability estimated via logistic regression. Black points at top and bottom of each plot represent single-cell measurements of cells which survived and died, respectively. Black points and error bars represent survival probability calculations from bins of 50 channels per cell. Blue points represent the survival probability for a given Shine-Dalgarno mutant. Horizontal error bars are the standard error of the mean with at least 25 measurements and vertical error bars signifies the uncertainty in the survival probability from observing n survival events out of N total measurements." id="fig:three_shock_group" />
<p class="caption">Figure 5: Coarse graining shock rates into different groups. Estimated survival probability curve for slow (A), intermediate (B), and fast (C) shock rates. (D) Estimated survival probability curve from pooling all data together, ignoring varying shock rates. Red shaded regions correspond to the 95% credible region of the survival probability estimated via logistic regression. Black points at top and bottom of each plot represent single-cell measurements of cells which survived and died, respectively. Black points and error bars represent survival probability calculations from bins of 50 channels per cell. Blue points represent the survival probability for a given Shine-Dalgarno mutant. Horizontal error bars are the standard error of the mean with at least 25 measurements and vertical error bars signifies the uncertainty in the survival probability from observing <span class="math inline">\(n\)</span> survival events out of <span class="math inline">\(N\)</span> total measurements.</p>
</div>
</p></div>
</section>
<section id="regression" class="shade-four">
  <div class="container"><header>
			<h2><a href="http://localhost:4000/supplement/2018/05/03/regression.html">Regression</a></h2></header><p><p>    In this work, we were interested in computing the survival probability under a large hypo-osmotic shock as a function of MscL channel number. As the channel copy number distributions for each Shine-Dalgarno sequence mutant were broad and overlapping, we chose to calculate the survival probability through logistic regression -- a method that requires no binning of the data providing the least biased estimate of survival probability. Logistic regression is a technique that has been used in medical statistics since the late 1950's to describe diverse phenomena such as dose response curves, criminal recidivism, and survival probabilities for patients after treatment <span class="citation">(4–6)</span>. It has also found much use in machine learning to tune a binary or categorical response given a continuous input <span class="citation">(7–9)</span>.</p>
<p>    In this section, we derive a statistical model for estimating the most-likely values for the coefficients <span class="math inline">\(\beta_0\)</span> and <span class="math inline">\(\beta_1\)</span> and use Bayes' theorem to provide an interpretation for the statistical meaning.</p>
<h3 id="bayesian-parameter-estimation-of-beta_0-and-beta_1">Bayesian parameter estimation of <span class="math inline">\(\beta_0\)</span> and <span class="math inline">\(\beta_1\)</span></h3>
<p>    The central challenge of this work is to estimate the probability of survival <span class="math inline">\(p_s\)</span> given only a measure of the total number of MscL channels in that cell. In other words, for a given measurement of <span class="math inline">\(N_c\)</span> channels, we want to know likelihood that a cell would survive an osmotic shock. Using Bayes' theorem, we can write a statistical model for the survival probability as <span id="eq:bayes_surv_prob"><span class="math display">\[
g(p_s\,\vert\, N_c) = {f(N_c\,\vert\, p_s)g(p_s) \over f(N_c)},
\qquad(17)\]</span></span></p>
<p>where <span class="math inline">\(g\)</span> and <span class="math inline">\(f\)</span> represent probability density functions over parameters and data, respectively. The posterior probability distribution <span class="math inline">\(g(p_s\,\vert\, N_c)\)</span> describes the probability of <span class="math inline">\(p_s\)</span> given a specific number of channels <span class="math inline">\(N_c\)</span>. This distribution is dependent on the likelihood of observing <span class="math inline">\(N_c\)</span> channels assuming a value of <span class="math inline">\(p_s\)</span> multiplied by all prior knowledge we have about knowing nothing about the data, <span class="math inline">\(g(s)\)</span>. The denominator <span class="math inline">\(f(N_c)\)</span> in Eq. 17 captures all knowledge we have about the available values of <span class="math inline">\(N_c\)</span>, knowing nothing about the true survival probability. As this term acts as a normalization constant, we will neglect it in the following calculations for convenience.</p>
<p>    To begin, we must come up with a statistical model that describes the experimental measurable in our experiment -- survival or death. As this is a binary response, we can consider each measurement as a Bernoulli trial with a probability of success matching our probability of survival <span class="math inline">\(p_s\)</span>, <span id="eq:bernoulli"><span class="math display">\[
f(s\, \vert\, p_s) = p_s^s(1 - p_s)^{1-s},
\qquad(18)\]</span></span> where <span class="math inline">\(s\)</span> is the binary response of <span class="math inline">\(1\)</span> or <span class="math inline">\(0\)</span> for survival and death, respectively. As is stated in the introduction to this section, we decided to use a logistic function to describe the survival probability. We assume that the log-odds of survival is linear with respect to the effective channel copy number <span class="math inline">\(N_c\)</span> as <span id="eq:logit"><span class="math display">\[
\log{p_s \over 1 - p_s} = \beta_0 + \beta_1 N_c,
\qquad(19)\]</span></span> where <span class="math inline">\(\beta_0\)</span> and <span class="math inline">\(\beta_1\)</span> are coefficients which describe the survival probability in the absence of channels and the increase in log-odds of survival conveyed by a single channel. The rationale behind this interpretation is presented in the following section, <em>A Bayesian interpretation of <span class="math inline">\(\beta_0\)</span> and <span class="math inline">\(\beta_1\)</span></em>. Using this assumption, we can solve for the survival probability <span class="math inline">\(p_s\)</span> as, <span id="eq:logistic_prob"><span class="math display">\[
p_s = {1 \over 1 + e^{-\beta_0 -\beta_1 N_c}}.
\qquad(20)\]</span></span></p>
<p>With a functional form for the survival probability, the likelihood stated in Eq. 17 can be restated as <span id="eq:bernoulli_likelihood"><span class="math display">\[
f(N_c, s\,\vert\,\beta_0,\beta_1) = \left({1 \over 1 + e^{-\beta_0 - \beta_1 N_c}}\right)^s\left(1 - {1 \over 1 + e^{-\beta_0 - \beta_1 N_c}}\right)^{1 - s}.
\qquad(21)\]</span></span> As we have now introduced two parameters, <span class="math inline">\(\beta_0\)</span>, and <span class="math inline">\(\beta_1\)</span>, we must provide some description of our prior knowledge regarding their values. As is typically the case, we know nothing about the values for <span class="math inline">\(\beta_0\)</span> and <span class="math inline">\(\beta_1\)</span>. These parameters are allowed to take any value, so long as it is a real number. Since all values are allowable, we can assume a flat distribution where any value has an equally likely probability. This value of this constant probability is not necessary for our calculation and is ignored. For a set of <span class="math inline">\(k\)</span> single-cell measurements, we can write the posterior probability distribution stated in Eq. 17 as <span id="eq:known_nc_post"><span class="math display">\[
g(\beta_0, \beta_1\,\vert\, N_c, s) = \prod\limits_{i=1}^n\left({1 \over 1 + e^{-\beta_0 - \beta_1 N_c^{(i)}}}\right)^{s^{(i)}}\left(1 - {1 \over 1 + e^{-\beta_0 - \beta_1 N_c^{(i)}}}\right)^{1 - s^{(i)}}
\qquad(22)\]</span></span></p>
<p>    Implicitly stated in Eq. 22 is absolute knowledge of the channel copy number <span class="math inline">\(N_c\)</span>. However, as is described in <em>Standard Candle Calibration</em>, we must convert from a measured areal sfGFP intensity <span class="math inline">\(I_A\)</span> to a effective channel copy number, <span id="eq:standard_candle"><span class="math display">\[
N_c = {I_A \tilde{\langle A \rangle} \over \tilde{\alpha}},
\qquad(23)\]</span></span> where <span class="math inline">\(\tilde{\langle A \rangle}\)</span> is the average cell area of the standard candle strain and <span class="math inline">\(\tilde{\alpha}\)</span> is the most-likely value for the calibration factor between arbitrary units and protein copy number. In <em>Standard Candle Calibration</em>, we detailed a process for generating an estimate for the most-likely value of <span class="math inline">\(\tilde{\langle A \rangle}\)</span> and <span class="math inline">\(\tilde{\alpha}\)</span>. Given these estimates, we can include an informative prior for each value. From the Markov chain Monte Carlo samples shown in Fig. 2, the posterior distribution for each parameter is approximately Gaussian. By approximating them as Gaussian distributions, we can assign an informative prior for each as <span id="eq:alpha_prior"><span class="math display">\[
g(\alpha\,\vert\,\tilde{\alpha}, \tilde{\sigma}_\alpha) \propto {1 \over \tilde{\sigma}_\alpha^k}\prod\limits_{i=1}^k\exp\left[-{(\alpha_i - \tilde{\alpha})^2 \over  2\tilde{\sigma}_\alpha^2}\right]
\qquad(24)\]</span></span> for the calibration factor for each cell and <span id="eq:area_prior"><span class="math display">\[
g(\langle A \rangle\,\vert\,\tilde{\langle A \rangle},\tilde{\sigma}_{\langle A \rangle}) = {1 \over \tilde{\sigma}_{\langle A \rangle}^k}\prod\limits_{i=1}^k\exp\left[-{(\langle A \rangle_i - \tilde{\langle A \rangle})^2 \over 2\tilde{\sigma}_{\langle A \rangle}^2}\right],
\qquad(25)\]</span></span> where <span class="math inline">\(\tilde{\sigma}_\alpha\)</span> and <span class="math inline">\(\tilde{\sigma}_{\langle A \rangle}\)</span> represent the variance from approximating each posterior as a Gaussian. The proportionality for each prior arises from the neglecting of normalization constants for notational convenience.</p>
<p>    Given Eq. 21 through Eq. 25, the complete posterior distribution for estimating the most likely values of <span class="math inline">\(\beta_0\)</span> and <span class="math inline">\(\beta_1\)</span> can be written as <span id="eq:logistic_posterior"><span class="math display">\[
\begin{aligned}
g(\beta_0, &amp;\beta_1\,\vert\,[I_A, s],\tilde{\langle A \rangle}, \tilde{\sigma}_{\langle A \rangle}, \tilde{\alpha}, \tilde{\sigma}_\alpha) \propto{1 \over (\tilde{\sigma}_\alpha\tilde{\sigma}_{\langle A \rangle})^k}\prod\limits_{i=1}^k\left(1 + \exp\left[-\beta_0 - \beta_1 {{I_A}_i \langle A \rangle_i \over \alpha_i}\right]\right)^{-s_i}\,\times\,\\
&amp;\left(1 - \left(1 + \exp\left[-\beta_0 - \beta_1 {{I_A}_i\langle A \rangle_i \over \alpha_i}\right]\right)^{-1}\right)^{1 - s_i}
\exp\left[-{(\langle A \rangle_i - \tilde{\langle A \rangle})^2 \over 2\tilde{\sigma}_{\langle A \rangle}} - {(\alpha_i - \tilde{\alpha})^2\over 2\tilde{\sigma}_\alpha^2}\right]
\end{aligned}.
\qquad(26)\]</span></span></p>
<p>As this posterior distribution is not solvable analytically, we used Markov chain Monte Carlo to draw samples out of this distribution, using the log of the effective channel number as described in the main text. The posterior distributions for <span class="math inline">\(\beta_0\)</span> and <span class="math inline">\(\beta_1\)</span> for both slow and fast shock rate data can be seen in Fig. 6</p>
<div class="figure">
<embed src="assets/figS4.png" id="fig:logistic_posterior_distributions" />
<p class="caption">Figure 6: Posterior distributions for logistic regression coefficients evaluated for fast and slow shock rates. (A) Kernel density estimates of the posterior distribution for <span class="math inline">\(\beta_0\)</span> for fast (blue) and slow (purple) shock rates. (B) Kernel density estimates of posterior distribution for <span class="math inline">\(\beta_1\)</span>.</p>
</div>
<h3 id="a-bayesian-interpretation-of-beta_0-and-beta_1">A Bayesian interpretation of <span class="math inline">\(\beta_0\)</span> and <span class="math inline">\(\beta_1\)</span></h3>
<p>The assumption of a linear relationship between the log-odds of survival and the predictor variable <span class="math inline">\(N_c\)</span> appears to be arbitrary and is presented without justification. However, this relationship is directly connected to the manner in which Bayes' theorem updates the posterior probability distribution upon the observation of new data. In following section, we will demonstrate this connection using the relationship between survival and channel copy number. However, this description is general and can be applied to any logistic regression model so long as the response variable is binary. This connection was shown briefly by Allen Downey in 2014 and has been expanded upon in this work <span class="citation">(10)</span>.</p>
<p>    The probability of observing a survival event <span class="math inline">\(s\)</span> given a measurement of <span class="math inline">\(N_c\)</span> channels can be stated using Bayes' theorem as <span id="eq:survival_bayes"><span class="math display">\[
g(s\,\vert\, N_c) = {f(N_c\,\vert\, s)g(s) \over f(N_c)}.
\qquad(27)\]</span></span> where <span class="math inline">\(g\)</span> and <span class="math inline">\(f\)</span> represent probability density functions over parameters and data respectively. The posterior distribution <span class="math inline">\(g(s\,\vert\, N_c)\)</span> is the quantity of interest and implicitly related to the probability of survival. The likelihood <span class="math inline">\(g(N_c\,\vert\, s)\)</span> tells us the probability of observing <span class="math inline">\(N_c\)</span> channels in this cell given that it survives. The quantity <span class="math inline">\(g(s)\)</span> captures all <em>a priori</em> knowledge we have regarding the probability of this cell surviving and the denominator <span class="math inline">\(f(N_c)\)</span> tells us the converse -- the probability of observing <span class="math inline">\(N_c\)</span> cells irrespective of the survival outcome.</p>
<p>    Proper calculation of Eq. 27 requires that we have knowledge of <span class="math inline">\(f(N_c)\)</span>, which is difficult to estimate. While we are able to give appropriate bounds on this term, such as a requirement of positivity and some knowledge of the maximum membrane packing density, it is not so obvious to determine the distribution between these bounds. Given this difficulty, it's easier to compute the odds of survival <span class="math inline">\(\mathcal{O}(s\,\vert\, N_c)\)</span>, the probability of survival <span class="math inline">\(s\)</span> relative to death <span class="math inline">\(d\)</span>, <span id="eq:odds_definition"><span class="math display">\[
\mathcal{O}(s\,\vert\, N_c) = {g(s\,\vert\,N_c) \over g(d\,\vert\, N_c)} = {f(N_c\,\vert\, s)g(s) \over f(N_c\,\vert\,d)g(d)},
\qquad(28)\]</span></span> where <span class="math inline">\(f(N_c)\)</span> is cancelled. The only stipulation on the possible value of the odds is that it must be a positive value. As we would like to equally weigh odds less than one as those of several hundred or thousand, it is more convenient to compute the log-odds, given as <span id="eq:log_odds"><span class="math display">\[
\log \mathcal{O}(s\,\vert\,N_c)= \log {g(s) \over g(d)} + \log {f(N_c \,\vert\, s )\over f(N_c\,\vert\, d)}.
\qquad(29)\]</span></span> Computing the log-transform reveals two interesting quantities. The first term is the ratio of the priors and tells us the <em>a priori</em> knowledge of the odds of survival irrespective of the number of channels. As we have no reason to think that survival is more likely than death, this ratio goes to unity. The second term is the log likelihood ratio and tells us how likely we are to observe a given channel copy number <span class="math inline">\(N_c\)</span> given the cell survives relative to when it dies.</p>
<p>    For each channel copy number, we can evaluate Eq. 29 to measure the log-odds of survival. If we start with zero channels per cell, we can write the log-odds of survival as <span id="eq:nc_0"><span class="math display">\[
\log \mathcal{O}(s\,\vert\,N_c=0) = \log {g(s) \over g(d)} + \log {f(N_c=0\,\vert\, s) \over f(N_c=0\,\vert\, d)}.
\qquad(30)\]</span></span> For a channel copy number of one, the odds of survival is <span id="eq:nc_1"><span class="math display">\[
\log \mathcal{O}(s\,\vert\,N_c=1) = \log{g(s) \over g(d)} + \log{f(N_c=1\,\vert\, s) \over f(N_c=1\,\vert\, d)}.
\qquad(31)\]</span></span> In both Eq. 30 and Eq. 31, the log of our <em>a priori</em> knowledge of survival versus death remains. The only factor that is changing is log likelihood ratio. We can be more general in our language and say that the log-odds of survival is increased by the difference in the log-odds conveyed by addition of a single channel. We can rewrite the log likelihood ratio in a more general form as <span id="eq:generalized_LLR"><span class="math display">\[
\log {f(N_c\, \vert\, s) \over f(N_c\,\vert\, d)} = \log{f(N_c = 0\,\vert\,s) \over f(N_c=0\,\vert\, d)} + N_c \left[\log{f(N_c=1 \,\vert\,s) \over f(N_c=1\,\vert\, d)} - \log{f(N_c=0\,\vert\, s) \over f(N_c=0\,\vert\, d)}\right],
\qquad(32)\]</span></span> where we are now only considering the case in which <span class="math inline">\(N_c \in [0, 1]\)</span>. The bracketed term in Eq. 32 is the log of the odds of survival given a single channel relative to the odds of survival given no channels. Mathematically, this odds-ratio can be expressed as <span id="eq:odds_ratio"><span class="math display">\[
\log\mathcal{OR}_{N_c}(s) = \log{{f(N_c=1\,\vert\,s)g(s)\over f(N_c=1\,\vert\,d)g(d)}\over {f(N_c=0\,\vert\,s)g(s)\over f(N_c=0\,\vert\,d)g(d)}} = \log{f(N_c=1\,\vert\,s) \over f(N_c=1\,\vert\,d)} - \log{f(N_c=0\,\vert\,s)\over f(N_c=0\,\vert\,d)} .
\qquad(33)\]</span></span> Eq. 33 is mathematically equivalent to the bracketed term shown in Eq. 32.</p>
<p>    We can now begin to staple these pieces together to arrive at an expression for the log odds of survival. Combining Eq. 32 with Eq. 29 yields <span id="eq:combined_v1"><span class="math display">\[
\log \mathcal{O}(s\,\vert\,N_c) = \log{g(s) \over g(d)} + \log {f(N_c=0\,\vert\, s) \over f(N_c=0\,\vert\, d)} + N_c\left[{f(N_c=1\,\vert\,s)\over f(N_c=1\,\vert\,d)} - \log{f(N_c=0\,\vert\,s) \over f(N_c=0\,\vert\,d)}\right].
\qquad(34)\]</span></span> Using our knowledge that the bracketed term is the log odds-ratio and the first two times represents the log-odds of survival with <span class="math inline">\(N_c=0\)</span>, we conclude with <span id="eq:bayesian_logit"><span class="math display">\[
\log\mathcal{O}(s\,\vert\,N_c) = \log \mathcal{O}(s\,\vert\,N_c=0) + N_c \log \mathcal{OR}_{N_c}(s).
\qquad(35)\]</span></span> This result can be directly compared to Eq. 1 presented in the main text, <span id="eq:"><span class="math display">\[
\log {p_s \over 1 - p_s} = \beta_0 + \beta_1 N_c,
\qquad(36)\]</span></span> which allows for an interpretation of the seemingly arbitrary coefficients <span class="math inline">\(\beta_0\)</span> and <span class="math inline">\(\beta_1\)</span>. The intercept term, <span class="math inline">\(\beta_0\)</span>, captures the log-odds of survival with no MscL channels. The slope, <span class="math inline">\(\beta_1\)</span>, describes the log odds-ratio of survival which a single channel relative to the odds of survival with no channels at all. While we have examined this considering only two possible channel copy numbers (<span class="math inline">\(1\)</span> and <span class="math inline">\(0\)</span>), the relationship between them is linear. We can therefore generalize this for any MscL copy number as the increase in the log-odds of survival is constant for addition of a single channel.</p>
<h3 id="other-properties-as-predictor-variables">Other properties as predictor variables</h3>
<p>     The previous two sections discuss in detail the logic and practice behind the application of logistic regression to cell survival data using only the effective channel copy number as the predictor of survival. However, there are a variety of properties that could rightly be used as predictor variables, such as cell area and shock rate. As is stipulated in our standard candle calibration, there should be no correlation between survival and cell area. Fig. 7A and B show the logistic regression performed on the cell area. We see for both slow and fast shock groups,there is little change in survival probability with changing cell area and the wide credible regions allow for both positive and negative correlation between survival and area. The appearance of a bottle neck in the notably wide credible regions is a result of a large fraction of the measurements being tightly distributed about a mean value. Fig. 7C shows the predicted survival probability as a function of the the shock rate. There is a slight decrease in survivability as a function of increasing shock rate, however the width of the credible region allows for slightly positive or slightly negative correlation. While we have presented logistic regression in this section as a one-dimensional method, Eq. 19 can be generalized to <span class="math inline">\(n\)</span> predictor variables <span class="math inline">\(x\)</span> as <span id="eq:generalized_logit"><span class="math display">\[
\log {p_s \over 1 - p_s} = \beta_0 + \sum\limits_{i}^n \beta_ix_i.
\qquad(37)\]</span></span> Using this generalization, we can use both shock rate and the effective channel copy number as predictor variables. The resulting two-dimensional surface of survival probability is shown in Fig. 7D. As is suggested by Fig. 7C, the magnitude of change in survivability as the shock rate is increased is smaller than that along increasing channel copy number, supporting our conclusion that for MscL alone, the copy number is the most important variable in determining survival.</p>
<div class="figure">
<img src="assets/figSX_alternative_predictors.png" alt="Figure 7: Survival probability estimation using alternative predictor variables. (A) Estimated survival probability as a function of cell area for the slow shock group. (B) Estimated survival probability as a function of cell area for the fast shock group. (C) Estimated survival probability as a function shock rate. Black points at top and bottom of plots represent single-cell measurements of cells who survived and perished, respectively. Shaded regions in (A) - (C) represent the 95% credible region. (D) Surface of estimated survival probability using both shock rate and effective channel number as predictor variables. Black points at left and right of plot represent single-cell measurements of cells which survived and died, respectively, sorted by shock rate. Points at top and bottom of plot represent survival and death sorted by their effective channel copy number. Labeled contours correspond to the survival probability." id="fig:alternative_predictor_variables" />
<p class="caption">Figure 7: Survival probability estimation using alternative predictor variables. (A) Estimated survival probability as a function of cell area for the slow shock group. (B) Estimated survival probability as a function of cell area for the fast shock group. (C) Estimated survival probability as a function shock rate. Black points at top and bottom of plots represent single-cell measurements of cells who survived and perished, respectively. Shaded regions in (A) - (C) represent the 95% credible region. (D) Surface of estimated survival probability using both shock rate and effective channel number as predictor variables. Black points at left and right of plot represent single-cell measurements of cells which survived and died, respectively, sorted by shock rate. Points at top and bottom of plot represent survival and death sorted by their effective channel copy number. Labeled contours correspond to the survival probability.</p>
</div>
<table style="width:100%;">
<caption><em>Escherichia coli</em> strains used in this work.</caption>
<colgroup>
<col width="17%" />
<col width="68%" />
<col width="13%" />
</colgroup>
<thead>
<tr class="header">
<th>Strain name</th>
<th>Genotype</th>
<th>Reference</th>
</tr>
</thead>
<tbody>
<tr class="odd">
<td>MJF641</td>
<td>Frag1, <em><span class="math inline">\(\Delta\)</span>mscL::cm, <span class="math inline">\(\Delta\)</span>mscS, <span class="math inline">\(\Delta\)</span>mscK::kan, <span class="math inline">\(\Delta\)</span>ybdG::apr, <span class="math inline">\(\Delta\)</span>ynaI, <span class="math inline">\(\Delta\)</span>yjeP, <span class="math inline">\(\Delta\)</span>ybiO, ycjM::Tn10</em></td>
<td><span class="citation">(11)</span></td>
</tr>
<tr class="even">
<td>MLG910</td>
<td>MG1655, <em><span class="math inline">\(\Delta\)</span>mscL ::<span class="math inline">\(\phi\)</span>mscL-sfGFP, <span class="math inline">\(\Delta\)</span>galK::kan, <span class="math inline">\(\Delta\)</span>lacI, <span class="math inline">\(\Delta\)</span>lacZY A</em></td>
<td><span class="citation">(1)</span></td>
</tr>
<tr class="odd">
<td>D6LG-Tn10</td>
<td>Frag1, <em><span class="math inline">\(\Delta\)</span>mscL ::<span class="math inline">\(\phi\)</span>mscL-sfGFP, <span class="math inline">\(\Delta\)</span>mscS, <span class="math inline">\(\Delta\)</span>mscK::kan, <span class="math inline">\(\Delta\)</span>ybdG::apr, <span class="math inline">\(\Delta\)</span>ynaI, <span class="math inline">\(\Delta\)</span>yjeP, <span class="math inline">\(\Delta\)</span>ybiO, ycjM::Tn10</em></td>
<td>This work</td>
</tr>
<tr class="even">
<td>D6LG (SD0)</td>
<td>Frag1, <em><span class="math inline">\(\Delta\)</span>mscL ::<span class="math inline">\(\phi\)</span>mscL-sfGFP, <span class="math inline">\(\Delta\)</span>mscS, <span class="math inline">\(\Delta\)</span>mscK::kan, <span class="math inline">\(\Delta\)</span>ybdG::apr, <span class="math inline">\(\Delta\)</span>ynaI, <span class="math inline">\(\Delta\)</span>yjeP, <span class="math inline">\(\Delta\)</span>ybiO</em></td>
<td>This work</td>
</tr>
<tr class="odd">
<td>XTL298</td>
<td>CC4231<em>, araD:: tetA-sacB-amp</em></td>
<td><span class="citation">(12)</span></td>
</tr>
<tr class="even">
<td>D6LTetSac</td>
<td>Frag1, <em>mscL-sfGFP:: tetA-sacB, <span class="math inline">\(\Delta\)</span>mscS, <span class="math inline">\(\Delta\)</span>mscK::kan, <span class="math inline">\(\Delta\)</span>ybdG::apr, <span class="math inline">\(\Delta\)</span>ynaI, <span class="math inline">\(\Delta\)</span>yjeP, <span class="math inline">\(\Delta\)</span>ybiO</em></td>
<td>This work</td>
</tr>
<tr class="odd">
<td>D6LG (SD1)</td>
<td>Frag1, <em><span class="math inline">\(\Delta\)</span>mscL ::<span class="math inline">\(\phi\)</span>mscL-sfGFP, <span class="math inline">\(\Delta\)</span>mscS, <span class="math inline">\(\Delta\)</span>mscK::kan, <span class="math inline">\(\Delta\)</span>ybdG::apr, <span class="math inline">\(\Delta\)</span>ynaI, <span class="math inline">\(\Delta\)</span>yjeP, <span class="math inline">\(\Delta\)</span>ybiO</em></td>
<td>This work</td>
</tr>
<tr class="even">
<td>D6LG (SD2)</td>
<td>Frag1, <em><span class="math inline">\(\Delta\)</span>mscL ::<span class="math inline">\(\phi\)</span>mscL-sfGFP, <span class="math inline">\(\Delta\)</span>mscS, <span class="math inline">\(\Delta\)</span>mscK::kan, <span class="math inline">\(\Delta\)</span>ybdG::apr, <span class="math inline">\(\Delta\)</span>ynaI, <span class="math inline">\(\Delta\)</span>yjeP, <span class="math inline">\(\Delta\)</span>ybiO</em></td>
<td>This work</td>
</tr>
<tr class="odd">
<td>D6LG (SD4)</td>
<td>Frag1, <em><span class="math inline">\(\Delta\)</span>mscL ::<span class="math inline">\(\phi\)</span>mscL-sfGFP, <span class="math inline">\(\Delta\)</span>mscS, <span class="math inline">\(\Delta\)</span>mscK::kan, <span class="math inline">\(\Delta\)</span>ybdG::apr, <span class="math inline">\(\Delta\)</span>ynaI, <span class="math inline">\(\Delta\)</span>yjeP, <span class="math inline">\(\Delta\)</span>ybiO</em></td>
<td>This work</td>
</tr>
<tr class="even">
<td>D6LG (SD6)</td>
<td>Frag1, <em><span class="math inline">\(\Delta\)</span>mscL ::<span class="math inline">\(\phi\)</span>mscL-sfGFP, <span class="math inline">\(\Delta\)</span>mscS, <span class="math inline">\(\Delta\)</span>mscK::kan, <span class="math inline">\(\Delta\)</span>ybdG::apr, <span class="math inline">\(\Delta\)</span>ynaI, <span class="math inline">\(\Delta\)</span>yjeP, <span class="math inline">\(\Delta\)</span>ybiO</em></td>
<td>This work</td>
</tr>
<tr class="odd">
<td>D6LG (12SD2)</td>
<td>Frag1, <em><span class="math inline">\(\Delta\)</span>mscL ::<span class="math inline">\(\phi\)</span>mscL-sfGFP, <span class="math inline">\(\Delta\)</span>mscS, <span class="math inline">\(\Delta\)</span>mscK::kan, <span class="math inline">\(\Delta\)</span>ybdG::apr, <span class="math inline">\(\Delta\)</span>ynaI, <span class="math inline">\(\Delta\)</span>yjeP, <span class="math inline">\(\Delta\)</span>ybiO</em></td>
<td>This work</td>
</tr>
<tr class="even">
<td>D6LG (16SD0)</td>
<td>Frag1, <em><span class="math inline">\(\Delta\)</span>mscL ::<span class="math inline">\(\phi\)</span>mscL-sfGFP, <span class="math inline">\(\Delta\)</span>mscS, <span class="math inline">\(\Delta\)</span>mscK::kan, <span class="math inline">\(\Delta\)</span>ybdG::apr, <span class="math inline">\(\Delta\)</span>ynaI, <span class="math inline">\(\Delta\)</span>yjeP, <span class="math inline">\(\Delta\)</span>ybiO</em></td>
<td>This work</td>
</tr>
</tbody>
</table>
<table>
<caption>Oligonucleotide sequences used in this work. Bold and italics correspond to Shine-Dalgarno sequence modifications and <code>AT</code> hairpin insertion modifications, respectively. Double bar <code>||</code> indicates a transposon insertion site.</caption>
<thead>
<tr class="header">
<th align="left">Primer Name</th>
<th align="left">Sequence (5' <span class="math inline">\(\rightarrow\)</span> 3')</th>
</tr>
</thead>
<tbody>
<tr class="odd">
<td align="left"><em>Tn10delR</em></td>
<td align="left"><code>taaagccaacggcatccaggcggacatactcagca||</code></td>
</tr>
<tr class="even">
<td align="left"></td>
<td align="left"><code>cctttcgcaaggtaacagagtaaaacatccaccat</code></td>
</tr>
<tr class="odd">
<td align="left"><em>MscLSPSac</em></td>
<td align="left"><code>gaaaatggcttaacatttgttagacttatggttgtcgg</code></td>
</tr>
<tr class="even">
<td align="left"></td>
<td align="left"><code>cttcat</code><strong><code>agggag</code></strong><code>TCCTAATTTTTGTTGACACTCTATC</code></td>
</tr>
<tr class="odd">
<td align="left"><em>MscLSPSacR</em></td>
<td align="left"><code>accacgttcccgcgcatcgcaaattcgcgaaat</code></td>
</tr>
<tr class="even">
<td align="left"></td>
<td align="left"><code>tctttaataatgctcatATCAAAGGGAAAACTGTCCATA</code></td>
</tr>
<tr class="odd">
<td align="left"><em>MscL-SD1R</em></td>
<td align="left"><code>atcgcaaattcgcgaaattctttaataatgctcat</code></td>
</tr>
<tr class="even">
<td align="left"></td>
<td align="left"><code>gttatt</code><strong><code>ctcctc</code></strong><code>atgaagccgacaaccataagtctaacaaa</code></td>
</tr>
<tr class="odd">
<td align="left"><em>MscL-SD2R</em></td>
<td align="left"><code>atcgcaaattcgcgaaattctttaataatgctcat</code><em><code>gttatt</code></em></td>
</tr>
<tr class="even">
<td align="left"></td>
<td align="left"><strong><code>tcccct</code></strong><code>atgaagccgacaaccataagtctaacaaa</code></td>
</tr>
<tr class="odd">
<td align="left"><em>MscL-SD4R</em></td>
<td align="left"><code>atcgcaaattcgcgaaattctttaataatgctcat</code></td>
</tr>
<tr class="even">
<td align="left"></td>
<td align="left"><em><code>gttatt</code></em> <strong><code>cctgct</code></strong><code>atgaagccgacaaccataagtctaacaaa</code></td>
</tr>
<tr class="odd">
<td align="left"><em>MscL-SD6R</em></td>
<td align="left"><code>atcgcaaattcgcgaaattctttaataatgctcat</code></td>
</tr>
<tr class="even">
<td align="left"></td>
<td align="left"><em><code>gttatt</code></em> <strong><code>gctcgt</code></strong><code>atgaagccgacaaccataagtctaacaaa</code></td>
</tr>
<tr class="odd">
<td align="left"><em>MscL-12SD2R</em></td>
<td align="left"><code>atcgcaaattcgcgaaattctttaataatgctcat</code></td>
</tr>
<tr class="even">
<td align="left"></td>
<td align="left"><em><code>atatatatatat</code></em> <strong><code>tcccct</code></strong><code>atgaagccgacaaccataagtctaacaaa</code></td>
</tr>
<tr class="odd">
<td align="left"><em>MscL-16SD0R</em></td>
<td align="left"><code>atcgcaaattcgcgaaattctttaataatgctcat</code></td>
</tr>
<tr class="even">
<td align="left"></td>
<td align="left"><em><code>atatatatatatatat</code></em> <strong><code>ctccct</code></strong><code>atgaagccgacaaccataagtctaacaaa</code></td>
</tr>
</tbody>
</table>
</p></div>
</section>
<section id="strains" class="shade-three">
  <div class="container"><header>
			<h2><a href="http://localhost:4000/supplement/2018/05/04/strains.html">Strains</a></h2></header><p><div id="refs" class="references">
<div id="ref-bialecka-fornal2012">
<p>1. Bialecka-Fornal M, Lee HJ, DeBerg HA, Gandhi CS, Phillips R. 2012. Single-Cell Census of Mechanosensitive Channels in Living Bacteria. PLoS ONE 7:e33077.</p>
</div>
<div id="ref-sivia2006">
<p>2. Sivia D, Skilling J. 2006. Data Analysis: A Bayesian TutorialSecond Edition. Oxford University Press, Oxford, New York.</p>
</div>
<div id="ref-bialecka-fornal2015">
<p>3. Bialecka-Fornal M, Lee HJ, Phillips R. 2015. The Rate of Osmotic Downshock Determines the Survival Probability of Bacterial Mechanosensitive Channel Mutants. Journal of Bacteriology 197:231–237.</p>
</div>
<div id="ref-anderson2003">
<p>4. Anderson RP, Jin R, Grunkemeier GL. 2003. Understanding logistic regression analysis in clinical reports: An introduction. The Annals of Thoracic Surgery 75:753–757.</p>
</div>
<div id="ref-mishra2016">
<p>5. Mishra V, Skotak M, Schuetz H, Heller A, Haorah J, Chandra N. 2016. Primary blast causes mild, moderate, severe and lethal TBI with increasing blast overpressures: Experimental rat injury model. Scientific Reports 6:26992.</p>
</div>
<div id="ref-stahler2013">
<p>6. Stahler GJ, Mennis J, Belenko S, Welsh WN, Hiller ML, Zajac G. 2013. Predicting Recidivism For Released State Prison Offenders. Criminal justice and behavior 40:690–711.</p>
</div>
<div id="ref-cheng2009">
<p>7. Cheng W, Hüllermeier E. 2009. Combining instance-based learning and logistic regression for multilabel classification. Machine Learning 76:211–225.</p>
</div>
<div id="ref-dreiseitl2002">
<p>8. Dreiseitl S, Ohno-Machado L. 2002. Logistic regression and artificial neural network classification models: A methodology review. Journal of Biomedical Informatics 35:352–359.</p>
</div>
<div id="ref-ng">
<p>9. Ng AY, Jordan MI. On Discriminative vs. Generative Classifiers: A comparison of logistic regression and naive Bayes 8.</p>
</div>
<div id="ref-downey2014">
<p>10. Downey A. 2014. Probably Overthinking It: Bayes’s theorem and logistic regression. Probably Overthinking It.</p>
</div>
<div id="ref-edwards2012">
<p>11. Edwards MD, Black S, Rasmussen T, Rasmussen A, Stokes NR, Stephen TL, Miller S, Booth IR. Jul-Aug 20122012. Characterization of three novel mechanosensitive channel activities in Escherichia coli. Channels (Austin) 6:272–81.</p>
</div>
<div id="ref-li2013">
<p>12. Li X-t, Thomason LC, Sawitzke JA, Costantino N, Court DL. 2013. Positive and negative selection using the tetA-sacB cassette: Recombineering and P1 transduction in Escherichia coli. Nucleic acids research 41:e204–e204.</p>
</div>
</div>
</body>
</html>
</p></div>
</section>
<section id="references" class="shade-two">
  <div class="container"><header>
			<h2><a href="http://localhost:4000/supplement/2018/05/05/references.html">References</a></h2></header><p></p></div>
</section>
</div>
	</article>
</div><!-- Footer -->
<div id="footer">
  
  <!-- Copyright -->
  <ul class="copyright">
   
</div></body>
</html>