{% raw %}<h3 id="bacterial-strains-and-growth-conditions">Bacterial strains and growth conditions</h3>
{% endraw %}{% raw %}<p>The bacterial strains are described in Table S1. The parent strain for the mutants used in this study was MJF641 <span class="citation">(<a href="#ref-edwards2012">5</a>)</span>, a strain which had all seven mechanosensitive channels deleted. The MscL-sfGFP coding region from MLG910 <span class="citation">(<a href="#ref-bialecka-fornal2012">3</a>)</span> was integrated into MJF641 by P1 transduction, creating the strain D6LG-Tn10. Selection pressure for MscL integration was created by incorporating an osmotic shock into the transduction protocol, which favored the survival of MscL-expressing stains relative to MJF641 by ~100-fold. Screening for integration candidates was based on fluorescence expression of plated colonies. Successful integration was verified by sequencing. Attempts to transduce RBS-modified MscL-sfGFP coding regions became increasingly inefficient as the targeted expression level of MscL was reduced. This was due to the decreasing fluorescence levels and survival rates of the integration candidates. Consequently, RBS modifications were made by inserting DNA oligos with lambda Red-mediated homologous recombination, i.e., recombineering [Sharan 2009]. The oligos had a designed mutation (Figure 2) flanked by ~25 base pairs that matched the targeted MscL region [Table S2]. A two-step recombineering process of selection followed by counter selection using a tetA-sacB gene fusion cassette <span class="citation">(<a href="#ref-li2013">42</a>)</span> was chosen because of its capabilities to integrate with efficiencies comparable to P1 transduction and not leave antibiotic resistance markers or scar sequences in the final strain. To prepare the strain D6LG-Tn10 for this scheme, the Tn10 transposon containing the tetA gene needed to be removed to avoid interference with the tetA-sacB cassette. Tn10 was removed from the middle of the ycjM gene with the primer Tn10delR (Table S2) by recombineering, creating the strain D6LG (SD0). Counter selection against the tetA gene was promoted by using agar media with fusaric acid <span class="citation">(<a href="#ref-li2013">42</a>, <a href="#ref-bochner1980">43</a>)</span>. The tetA-sacB cassette was PCR amplified out of the strain XTL298 using primers MscLSPSac and MscLSPSacR (Table S2). The cassette was integrated in place of the spacer region in front of the MscL start codon of D6LG (SD0) by recombineering, creating the intermediate strain D6LTetSac. Positive selection for cassette integration was provided by agar media with tetracycline. Finally, the RBS modifying oligos were integrated into place by replacing the tetA-sacB cassette by recombineering. Counter selection against both tetA and sacB was ensured by using agar media with fusaric acid and sucrose <span class="citation">(<a href="#ref-li2013">42</a>)</span>, creating the Shine-Dalgarno mutant strains used in this work.</p>
{% endraw %}{% raw %}<p>Strain cultures were grown in 5 mL of LB-Lennox media with antibiotic (apramycin) overnight at 37°C. The next day, 50 µL of overnight culture was inoculated into 5 mL of LB-Lenox with antibiotic and the culture was grown to OD<sub>600nm</sub> ~0.25. Subsequently, 500 µL of that culture was inoculated into 5 mL of LB-Lennox supplemented with 500mM of NaCl and the culture was regrown to OD<sub>600nm</sub> ~0.25. A 1 mL aliquot was taken and used to load the flow cell.</p>
{% endraw %}{% raw %}<h3 id="flow-cell">Flow cell</h3>
{% endraw %}{% raw %}<p>        All experiments were conducted in a home-made flow cell as is shown in Fig. 3A. This flow cell has two inlets which allow media of different osmolarity to be exchanged over the course of the experiment. The imaging region is approximately 10 mm wide and 100 <span class="math inline">\(\mu\)</span>m in depth. All imaging took place within 1 – 2 cm of the outlet to avoid imaging cells within a non-uniform gradient of osmolarity. The interior of the flow cell was functionalized with a 1:400 dilution of polyethylamine prior to addition of cells with the excess washed away with water. A dilute cell suspension in LB Lennox with 500 mM NaCl was loaded into one inlet while the other was connected to a vial of LB medium with no NaCl. This hypotonic medium was clamped during the loading of the cells.</p>
{% endraw %}{% raw %}<p>       Once the cells had adhered to the polyethylamine coated surface, the excess cells were washed away with the 500 mM NaCl growth medium followed by a small (~20 <span class="math inline">\(\mu\)</span>L) air bubble. This air bubble forced the cells to lay flat against the imaging surface, improving the time-lapse imaging. Over the observation period, cells not exposed to an osmotic shock were able to grow for 4 – 6 divisions, showing that the flow cell does not directly impede cell growth.</p>
{% endraw %}{% raw %}<h3 id="imaging-conditions">Imaging conditions</h3>
{% endraw %}{% raw %}<p>       All imaging was performed in a flow cell held at 30°C on a Nikon Ti-Eclipse microscope outfitted with a Perfect Focus system enclosed in a Haison environmental chamber (approximately 1°C regulation efficiency). The microscope was equipped with a 488 nm laser excitation source (CrystaLaser) and a 520/35 laser optimized filter set (Semrock). The images were collected on an Andor Xion +897 EMCCD camera and all microscope and acquisition operations were controlled via the open source <span class="math inline">\(\mu\)</span>Manager microscope control software (27). Once cells were securely mounted onto the surface of the glass coverslip, between 15 and 20 positions containing 5 to 10 cells were marked and the coordinates recorded. At each position, a phase contrast and GFP fluorescence image was acquired for segmentation and subsequent measurement of channel copy number. To perform the osmotic shock, LB media containing no NaCl was pulled into the flow cell through a syringe pump. To monitor the media exchange, both the high salt and no salt LB media were supplemented with a low-affinity version of the calcium-sensitive dye Rhod-2 (250 nM; TEF Labs) which fluoresces when bound to Ca<sup>2+</sup>. The no salt medium was also supplemented with 1<span class="math inline">\(\mu\)</span>M CaCl<sub>2</sub> to make the media mildly fluorescent and the exchange rate was calculated by measuring the fluorescence increase across an illuminated section of one of the positions. These images were collected in real time for the duration of the shock. The difference in measured fluorescence between the pre-shock images and those at the end of the shock set the scale of a 500 mM NaCl down shock. The rate was calculated by fitting a line to the middle region of this trace. Further details regarding this procedure can be found in Bialecka-Fornal, Lee, and Phillips, 2015 <span class="citation">(<a href="#ref-bialecka-fornal2015">4</a>)</span>.</p>
{% endraw %}{% raw %}<h3 id="image-processing">Image Processing</h3>
{% endraw %}{% raw %}<p>       Images were processed using a combination of automated and manual methods. First, expression of MscL was measured via segmenting individual cells or small clusters of cells in phase contrast and computing the mean pixel value of the fluorescence image for each segmented object. The fluorescence images were passed through several filtering operations which reduced high-frequency noise as well as corrected for uneven illumination of the excitation wavelength.</p>
{% endraw %}{% raw %}<p>       Survival or death classification was performed manually using the CellProfiler plugin for ImageJ software (NIH). A survivor was defined as a cell which was able to undergo two division events after the osmotic down shock. Cells which detached from the surface during the post-shock growth phase or those which became indistinguishable from other cells due to clustering were not counted as survival or death and were removed from the dataset completely. A region of the cell was manually marked with 1.0 (survival) or 0.0 (death) by clicking on the image. The <code>xy</code> coordinates of the click as well as the assigned value were saved as an <code>.xml</code> file for that position.</p>
{% endraw %}{% raw %}<p>       The connection between the segmented cells and their corresponding manual markers was automated. As the manual markings were made on the first phase contrast image after the osmotic shock, small shifts in the positions of the cell made one-to-one mapping with the segmentation mask non-trivial. The linkages between segmented cell and manual marker were made by computing all pairwise distances between the manual marker and the segmented cell centroid, taking the shortest distance as the true pairing. The linkages were then inspected manually and incorrect mappings were corrected as necessary.</p>
{% endraw %}{% raw %}<p>       All relevant statistics about the segmented objects as well as the sample identity, date of acquisition, osmotic shock rate, and camera exposure time were saved as <code>.csv</code> files for each individual experiment. A more in-depth description of the segmentation procedure as well as the relevant code can be accessed as a Jupyter Notebook at (<code>http://rpgroup.caltech.edu/mscl_survival</code>).</p>
{% endraw %}{% raw %}<h3 id="calculation-of-effective-channel-copy-number">Calculation of effective channel copy number</h3>
{% endraw %}{% raw %}<p>       To compute the MscL channel copy number, we relied on measuring the fluorescence level of a bacterial strain in which the mean MscL channel copy number was known via fluorescence microscopy <span class="citation">(<a href="#ref-bialecka-fornal2012">3</a>)</span>. <em>E. coli</em> strain MLG910, which expresses the MscL-sfGFP fusion protein from the wild-type SD sequence, was grown under identical conditions to those described in Bialecka-Fornal et al. 2015 in M9 minimal medium supplemented with 0.5% glucose to an OD<sub>600nm</sub> of ~0.3. The cells were then diluted ten fold and immobilized on a rigid 2% agarose substrate and placed onto a glass bottom petri dish and imaged in the same conditions as described previously.</p>
{% endraw %}{% raw %}<p>     Images were taken of six biological replicates of MLG910 and were processed identically to those in the osmotic shock experiments. A calibration factor between the average cell fluorescence level and mean MscL copy number was then computed. We assumed that all measured fluorescence (after filtering and background subtraction) was derived from the MscL-sfGFP fusion, <span id="eq:ian"><span class="math display">\[
{% endraw %}{% raw %}\langle I_\text{tot}\rangle = \alpha \langle N \rangle,
{% endraw %}{% raw %}\qquad(3)\]</span></span> in which <span class="math inline">\(\alpha\)</span> is the calibration factor and <span class="math inline">\(\langle N \rangle\)</span> is the mean cellular MscL-sfGFP copy number as reported in Bialecka-Fornal et al. 2012 <span class="citation">(<a href="#ref-bialecka-fornal2012">3</a>)</span>. To correct for errors in segmentation, the intensity was computed as an areal density <span class="math inline">\(\langle I_A \rangle\)</span> and was multiplied by the average cell area <span class="math inline">\(\langle A \rangle\)</span> of the population. The calibration factor was therefore computed as <span id="eq:calibration_factor"><span class="math display">\[
{% endraw %}{% raw %}\alpha = {\langle I_A \rangle \langle A \rangle \over \langle N \rangle}.
{% endraw %}{% raw %}\qquad(4)\]</span></span></p>
{% endraw %}{% raw %}<p>   We used Bayesian inferential methods to compute this calibration factor taking measurement error and replicate-to-replicate variation into account. The resulting average cell area and calibration factor was used to convert the measured cell intensities from the osmotic shock experiments to cell copy number. The details of this inference are described in depth in the supplemental information (<em>Standard Candle Calibration</em>).</p>
{% endraw %}{% raw %}<h3 id="logistic-regression">Logistic regression</h3>
{% endraw %}{% raw %}<p>       We used Bayesian inferential methods to find the most probable values of the coefficients <span class="math inline">\(\beta_0\)</span> and <span class="math inline">\(\beta_1\)</span> and the appropriate credible regions and is described in detail in the supplemental information (<em>Logistic Regression</em>). Briefly, we used Markov chain Monte Carlo (MCMC) to sample from the log posterior distribution and took the most probable value as the mean of the samples for each parameter. The MCMC was performed using the Stan probabilistic programming language <span class="citation">(<a href="#ref-carpenter2017">44</a>)</span> and all models can be found on the GitHub repository (<code>http://github.com/rpgroup-pboc/mscl_survival</code>).</p>
{% endraw %}{% raw %}<h3 id="calculation-of-survival-probability-error">Calculation of survival probability error</h3>
{% endraw %}{% raw %}<p>    The vertical error bars for the points shown in Fig. 5 represent our uncertainty in the survival probability given our measurement of <span class="math inline">\(n\)</span> survivors out of a total <span class="math inline">\(N\)</span> single-cell measurements. The probability distribution of the survival probability <span class="math inline">\(p_s\)</span> given these measurements can be written using Bayes' theorem as <span id="eq:probability_bayes"><span class="math display">\[
{% endraw %}{% raw %}g(p_s\,\vert\, n, N) = {f(n\,\vert\,p_s, N)g(p_s) \over f(n\,\vert\, N)},
{% endraw %}{% raw %}\qquad(5)\]</span></span> where <span class="math inline">\(g\)</span> and <span class="math inline">\(f\)</span> represent probability density functions over parameters and data, respectively. The likelihood <span class="math inline">\(f(n\,\vert p_s, N)\)</span> represents the probability of measuring <span class="math inline">\(n\)</span> survival events, given a total of <span class="math inline">\(N\)</span> measurements each with a probability of survival <span class="math inline">\(p_s\)</span>. This matches the story for the Binomial distribution and can be written as <span id="eq:binomial"><span class="math display">\[
{% endraw %}{% raw %}f(n\,\vert\,p_s, N) = {N! \over n!(N - n)!}p_s^n(1 - p_s)^{N - n}.
{% endraw %}{% raw %}\qquad(6)\]</span></span> To maintain maximal ignorance we can assume that any value for <span class="math inline">\(p_s\)</span> is valid, such that is in the range [0, 1]. This prior knowledge, represented by <span class="math inline">\(g(p_s)\)</span>, can be written as <span id="eq:uniform_prob"><span class="math display">\[
{% endraw %}{% raw %}g(p_s) = \begin{cases}1 &amp; 0\leq p_s\leq 1 \\
{% endraw %}{% raw %}0 &amp; \text{otherwise} \end{cases}.
{% endraw %}{% raw %}\qquad(7)\]</span></span> We can also assume maximal ignorance for the total number of survival events we could measure given <span class="math inline">\(N\)</span> observations, <span class="math inline">\(f(n\, \vert\, N)\)</span>. Assuming all observations are equally likely, this can be written as <span id="eq:evidence"><span class="math display">\[
{% endraw %}{% raw %}f(n\,\vert\, N) = {1 \over N + 1}
{% endraw %}{% raw %}\qquad(8)\]</span></span> where the addition of one comes from the possibility of observing zero survival events. Combining Eqns. 6-8, the posterior distribution <span class="math inline">\(g(p_s\,\vert\, n, N)\)</span> is <span id="eq:probability_posterior"><span class="math display">\[
{% endraw %}{% raw %}g(p_s\,\vert\, n, N) = {(N+1)! \over n!(N - n)!}p_s^{n}(1 - p_s)^{N - n}.
{% endraw %}{% raw %}\qquad(9)\]</span></span></p>
{% endraw %}{% raw %}<p>     The most probable value of <span class="math inline">\(p_s\)</span>, where the posterior probability distribution given by Eq. 9 is maximized, can be found by computing the point at which derivative of the log posterior with respect to <span class="math inline">\(p_s\)</span> goes to zero, <span id="eq:deriv_ps"><span class="math display">\[
{% endraw %}{% raw %}{d\log g(p_s\,\vert\,n, N) \over d p_s} = {n \over p_s} - {N - n  \over 1 - p_s} = 0.
{% endraw %}{% raw %}\qquad(10)\]</span></span> Solving Eq. 10 for <span class="math inline">\(p_s\)</span> gives the most likely value for the probability, <span id="eq:most_prob_prob"><span class="math display">\[
{% endraw %}{% raw %}p_s^* = {n \over N}.
{% endraw %}{% raw %}\qquad(11)\]</span></span> So long as <span class="math inline">\(N &gt;&gt; np_s^*\)</span>, Eq. 9 can be approximated as a Gaussian distribution with a mean <span class="math inline">\(p_s^*\)</span> and a variance <span class="math inline">\(\sigma_{p_s}^2\)</span>. By definition, the variance of a Gaussian distribution is computed as the negative reciprocal of the second derivative of the log posterior evaluated at <span class="math inline">\(p_s = p_s^*\)</span>, <span id="eq:variance_def"><span class="math display">\[
{% endraw %}{% raw %}\sigma_{p_s}^2 = - \left({d^2 \log g(p_s\,\vert\, n, N) \over dp_s^2}\Bigg\vert_{p_s=p_s^*}\right)^{-1}.
{% endraw %}{% raw %}\qquad(12)\]</span></span> Evaluating Eq. 12 yields <span id="eq:prob_variance"><span class="math display">\[
{% endraw %}{% raw %}\sigma_{p_s}^2 = {n(N-n)\over N^3}.
{% endraw %}{% raw %}\qquad(13)\]</span></span> Given Eq. 11 and Eq. 13, the most-likely survival probability and estimate of the uncertainty can be expressed as <span id="eq:"><span class="math display">\[
{% endraw %}{% raw %}p_s = p_s^* \pm \sigma_{p_s}.
{% endraw %}{% raw %}\qquad(14)\]</span></span></p>
{% endraw %}{% raw %}<h3 id="data-and-software-availability">Data and software availability</h3>
{% endraw %}{% raw %}<p>       All raw image data is freely available and is stored on the CaltechDATA Research Data Repository <span class="citation">(<a href="#ref-chure2018a">45</a>)</span>. The raw Markov chain Monte Carlo samples are stored as <code>.csv</code> files on CaltechDATA <span class="citation">(<a href="#ref-chure2018">46</a>)</span>. All processed experimental data, Python, and Stan code used in this work are freely available through our GitHub repository (<code>http://github.com/rpgroup-pboc/mscl_survival</code>)<span class="citation">(<a href="#ref-chure2018b">47</a>)</span> accessible through DOI: 10.5281/zenodo.1252524. The scientific community is invited to fork our repository and open constructive issues.</p>
{% endraw %}