{% raw %}<p>    In this work, we were interested in computing the survival probability under a large hypo-osmotic shock as a function of MscL channel number. As the channel copy number distributions for each Shine-Dalgarno sequence mutant were broad and overlapping, we chose to calculate the survival probability through logistic regression -- a method that requires no binning of the data providing the least biased estimate of survival probability. Logistic regression is a technique that has been used in medical statistics since the late 1950's to describe diverse phenomena such as dose response curves, criminal recidivism, and survival probabilities for patients after treatment <span class="citation">(4–6)</span>. It has also found much use in machine learning to tune a binary or categorical response given a continuous input <span class="citation">(7–9)</span>.</p>
{% endraw %}{% raw %}<p>    In this section, we derive a statistical model for estimating the most-likely values for the coefficients <span class="math inline">\(\beta_0\)</span> and <span class="math inline">\(\beta_1\)</span> and use Bayes' theorem to provide an interpretation for the statistical meaning.</p>
{% endraw %}{% raw %}<h3 id="bayesian-parameter-estimation-of-beta_0-and-beta_1">Bayesian parameter estimation of <span class="math inline">\(\beta_0\)</span> and <span class="math inline">\(\beta_1\)</span></h3>
{% endraw %}{% raw %}<p>    The central challenge of this work is to estimate the probability of survival <span class="math inline">\(p_s\)</span> given only a measure of the total number of MscL channels in that cell. In other words, for a given measurement of <span class="math inline">\(N_c\)</span> channels, we want to know likelihood that a cell would survive an osmotic shock. Using Bayes' theorem, we can write a statistical model for the survival probability as <span id="eq:bayes_surv_prob"><span class="math display">\[
{% endraw %}{% raw %}g(p_s\,\vert\, N_c) = {f(N_c\,\vert\, p_s)g(p_s) \over f(N_c)},
{% endraw %}{% raw %}\qquad(17)\]</span></span></p>
{% endraw %}{% raw %}<p>where <span class="math inline">\(g\)</span> and <span class="math inline">\(f\)</span> represent probability density functions over parameters and data, respectively. The posterior probability distribution <span class="math inline">\(g(p_s\,\vert\, N_c)\)</span> describes the probability of <span class="math inline">\(p_s\)</span> given a specific number of channels <span class="math inline">\(N_c\)</span>. This distribution is dependent on the likelihood of observing <span class="math inline">\(N_c\)</span> channels assuming a value of <span class="math inline">\(p_s\)</span> multiplied by all prior knowledge we have about knowing nothing about the data, <span class="math inline">\(g(s)\)</span>. The denominator <span class="math inline">\(f(N_c)\)</span> in Eq. 17 captures all knowledge we have about the available values of <span class="math inline">\(N_c\)</span>, knowing nothing about the true survival probability. As this term acts as a normalization constant, we will neglect it in the following calculations for convenience.</p>
{% endraw %}{% raw %}<p>    To begin, we must come up with a statistical model that describes the experimental measurable in our experiment -- survival or death. As this is a binary response, we can consider each measurement as a Bernoulli trial with a probability of success matching our probability of survival <span class="math inline">\(p_s\)</span>, <span id="eq:bernoulli"><span class="math display">\[
{% endraw %}{% raw %}f(s\, \vert\, p_s) = p_s^s(1 - p_s)^{1-s},
{% endraw %}{% raw %}\qquad(18)\]</span></span> where <span class="math inline">\(s\)</span> is the binary response of <span class="math inline">\(1\)</span> or <span class="math inline">\(0\)</span> for survival and death, respectively. As is stated in the introduction to this section, we decided to use a logistic function to describe the survival probability. We assume that the log-odds of survival is linear with respect to the effective channel copy number <span class="math inline">\(N_c\)</span> as <span id="eq:logit"><span class="math display">\[
{% endraw %}{% raw %}\log{p_s \over 1 - p_s} = \beta_0 + \beta_1 N_c,
{% endraw %}{% raw %}\qquad(19)\]</span></span> where <span class="math inline">\(\beta_0\)</span> and <span class="math inline">\(\beta_1\)</span> are coefficients which describe the survival probability in the absence of channels and the increase in log-odds of survival conveyed by a single channel. The rationale behind this interpretation is presented in the following section, <em>A Bayesian interpretation of <span class="math inline">\(\beta_0\)</span> and <span class="math inline">\(\beta_1\)</span></em>. Using this assumption, we can solve for the survival probability <span class="math inline">\(p_s\)</span> as, <span id="eq:logistic_prob"><span class="math display">\[
{% endraw %}{% raw %}p_s = {1 \over 1 + e^{-\beta_0 -\beta_1 N_c}}.
{% endraw %}{% raw %}\qquad(20)\]</span></span></p>
{% endraw %}{% raw %}<p>With a functional form for the survival probability, the likelihood stated in Eq. 17 can be restated as <span id="eq:bernoulli_likelihood"><span class="math display">\[
{% endraw %}{% raw %}f(N_c, s\,\vert\,\beta_0,\beta_1) = \left({1 \over 1 + e^{-\beta_0 - \beta_1 N_c}}\right)^s\left(1 - {1 \over 1 + e^{-\beta_0 - \beta_1 N_c}}\right)^{1 - s}.
{% endraw %}{% raw %}\qquad(21)\]</span></span> As we have now introduced two parameters, <span class="math inline">\(\beta_0\)</span>, and <span class="math inline">\(\beta_1\)</span>, we must provide some description of our prior knowledge regarding their values. As is typically the case, we know nothing about the values for <span class="math inline">\(\beta_0\)</span> and <span class="math inline">\(\beta_1\)</span>. These parameters are allowed to take any value, so long as it is a real number. Since all values are allowable, we can assume a flat distribution where any value has an equally likely probability. This value of this constant probability is not necessary for our calculation and is ignored. For a set of <span class="math inline">\(k\)</span> single-cell measurements, we can write the posterior probability distribution stated in Eq. 17 as <span id="eq:known_nc_post"><span class="math display">\[
{% endraw %}{% raw %}g(\beta_0, \beta_1\,\vert\, N_c, s) = \prod\limits_{i=1}^n\left({1 \over 1 + e^{-\beta_0 - \beta_1 N_c^{(i)}}}\right)^{s^{(i)}}\left(1 - {1 \over 1 + e^{-\beta_0 - \beta_1 N_c^{(i)}}}\right)^{1 - s^{(i)}}
{% endraw %}{% raw %}\qquad(22)\]</span></span></p>
{% endraw %}{% raw %}<p>    Implicitly stated in Eq. 22 is absolute knowledge of the channel copy number <span class="math inline">\(N_c\)</span>. However, as is described in <em>Standard Candle Calibration</em>, we must convert from a measured areal sfGFP intensity <span class="math inline">\(I_A\)</span> to a effective channel copy number, <span id="eq:standard_candle"><span class="math display">\[
{% endraw %}{% raw %}N_c = {I_A \tilde{\langle A \rangle} \over \tilde{\alpha}},
{% endraw %}{% raw %}\qquad(23)\]</span></span> where <span class="math inline">\(\tilde{\langle A \rangle}\)</span> is the average cell area of the standard candle strain and <span class="math inline">\(\tilde{\alpha}\)</span> is the most-likely value for the calibration factor between arbitrary units and protein copy number. In <em>Standard Candle Calibration</em>, we detailed a process for generating an estimate for the most-likely value of <span class="math inline">\(\tilde{\langle A \rangle}\)</span> and <span class="math inline">\(\tilde{\alpha}\)</span>. Given these estimates, we can include an informative prior for each value. From the Markov chain Monte Carlo samples shown in Fig. 2, the posterior distribution for each parameter is approximately Gaussian. By approximating them as Gaussian distributions, we can assign an informative prior for each as <span id="eq:alpha_prior"><span class="math display">\[
{% endraw %}{% raw %}g(\alpha\,\vert\,\tilde{\alpha}, \tilde{\sigma}_\alpha) \propto {1 \over \tilde{\sigma}_\alpha^k}\prod\limits_{i=1}^k\exp\left[-{(\alpha_i - \tilde{\alpha})^2 \over  2\tilde{\sigma}_\alpha^2}\right]
{% endraw %}{% raw %}\qquad(24)\]</span></span> for the calibration factor for each cell and <span id="eq:area_prior"><span class="math display">\[
{% endraw %}{% raw %}g(\langle A \rangle\,\vert\,\tilde{\langle A \rangle},\tilde{\sigma}_{\langle A \rangle}) = {1 \over \tilde{\sigma}_{\langle A \rangle}^k}\prod\limits_{i=1}^k\exp\left[-{(\langle A \rangle_i - \tilde{\langle A \rangle})^2 \over 2\tilde{\sigma}_{\langle A \rangle}^2}\right],
{% endraw %}{% raw %}\qquad(25)\]</span></span> where <span class="math inline">\(\tilde{\sigma}_\alpha\)</span> and <span class="math inline">\(\tilde{\sigma}_{\langle A \rangle}\)</span> represent the variance from approximating each posterior as a Gaussian. The proportionality for each prior arises from the neglecting of normalization constants for notational convenience.</p>
{% endraw %}{% raw %}<p>    Given Eq. 21 through Eq. 25, the complete posterior distribution for estimating the most likely values of <span class="math inline">\(\beta_0\)</span> and <span class="math inline">\(\beta_1\)</span> can be written as <span id="eq:logistic_posterior"><span class="math display">\[
{% endraw %}{% raw %}\begin{aligned}
{% endraw %}{% raw %}g(\beta_0, &amp;\beta_1\,\vert\,[I_A, s],\tilde{\langle A \rangle}, \tilde{\sigma}_{\langle A \rangle}, \tilde{\alpha}, \tilde{\sigma}_\alpha) \propto{1 \over (\tilde{\sigma}_\alpha\tilde{\sigma}_{\langle A \rangle})^k}\prod\limits_{i=1}^k\left(1 + \exp\left[-\beta_0 - \beta_1 {{I_A}_i \langle A \rangle_i \over \alpha_i}\right]\right)^{-s_i}\,\times\,\\
{% endraw %}{% raw %}&amp;\left(1 - \left(1 + \exp\left[-\beta_0 - \beta_1 {{I_A}_i\langle A \rangle_i \over \alpha_i}\right]\right)^{-1}\right)^{1 - s_i}
{% endraw %}{% raw %}\exp\left[-{(\langle A \rangle_i - \tilde{\langle A \rangle})^2 \over 2\tilde{\sigma}_{\langle A \rangle}} - {(\alpha_i - \tilde{\alpha})^2\over 2\tilde{\sigma}_\alpha^2}\right]
{% endraw %}{% raw %}\end{aligned}.
{% endraw %}{% raw %}\qquad(26)\]</span></span></p>
{% endraw %}{% raw %}<p>As this posterior distribution is not solvable analytically, we used Markov chain Monte Carlo to draw samples out of this distribution, using the log of the effective channel number as described in the main text. The posterior distributions for <span class="math inline">\(\beta_0\)</span> and <span class="math inline">\(\beta_1\)</span> for both slow and fast shock rate data can be seen in Fig. 6</p>
{% endraw %}{% raw %}<div class="figure">
{% endraw %}{% raw %}<embed src="assets/figS4.png" id="fig:logistic_posterior_distributions" />
{% endraw %}{% raw %}<p class="caption">Figure 6: Posterior distributions for logistic regression coefficients evaluated for fast and slow shock rates. (A) Kernel density estimates of the posterior distribution for <span class="math inline">\(\beta_0\)</span> for fast (blue) and slow (purple) shock rates. (B) Kernel density estimates of posterior distribution for <span class="math inline">\(\beta_1\)</span>.</p>
{% endraw %}{% raw %}</div>
{% endraw %}{% raw %}<h3 id="a-bayesian-interpretation-of-beta_0-and-beta_1">A Bayesian interpretation of <span class="math inline">\(\beta_0\)</span> and <span class="math inline">\(\beta_1\)</span></h3>
{% endraw %}{% raw %}<p>The assumption of a linear relationship between the log-odds of survival and the predictor variable <span class="math inline">\(N_c\)</span> appears to be arbitrary and is presented without justification. However, this relationship is directly connected to the manner in which Bayes' theorem updates the posterior probability distribution upon the observation of new data. In following section, we will demonstrate this connection using the relationship between survival and channel copy number. However, this description is general and can be applied to any logistic regression model so long as the response variable is binary. This connection was shown briefly by Allen Downey in 2014 and has been expanded upon in this work <span class="citation">(10)</span>.</p>
{% endraw %}{% raw %}<p>    The probability of observing a survival event <span class="math inline">\(s\)</span> given a measurement of <span class="math inline">\(N_c\)</span> channels can be stated using Bayes' theorem as <span id="eq:survival_bayes"><span class="math display">\[
{% endraw %}{% raw %}g(s\,\vert\, N_c) = {f(N_c\,\vert\, s)g(s) \over f(N_c)}.
{% endraw %}{% raw %}\qquad(27)\]</span></span> where <span class="math inline">\(g\)</span> and <span class="math inline">\(f\)</span> represent probability density functions over parameters and data respectively. The posterior distribution <span class="math inline">\(g(s\,\vert\, N_c)\)</span> is the quantity of interest and implicitly related to the probability of survival. The likelihood <span class="math inline">\(g(N_c\,\vert\, s)\)</span> tells us the probability of observing <span class="math inline">\(N_c\)</span> channels in this cell given that it survives. The quantity <span class="math inline">\(g(s)\)</span> captures all <em>a priori</em> knowledge we have regarding the probability of this cell surviving and the denominator <span class="math inline">\(f(N_c)\)</span> tells us the converse -- the probability of observing <span class="math inline">\(N_c\)</span> cells irrespective of the survival outcome.</p>
{% endraw %}{% raw %}<p>    Proper calculation of Eq. 27 requires that we have knowledge of <span class="math inline">\(f(N_c)\)</span>, which is difficult to estimate. While we are able to give appropriate bounds on this term, such as a requirement of positivity and some knowledge of the maximum membrane packing density, it is not so obvious to determine the distribution between these bounds. Given this difficulty, it's easier to compute the odds of survival <span class="math inline">\(\mathcal{O}(s\,\vert\, N_c)\)</span>, the probability of survival <span class="math inline">\(s\)</span> relative to death <span class="math inline">\(d\)</span>, <span id="eq:odds_definition"><span class="math display">\[
{% endraw %}{% raw %}\mathcal{O}(s\,\vert\, N_c) = {g(s\,\vert\,N_c) \over g(d\,\vert\, N_c)} = {f(N_c\,\vert\, s)g(s) \over f(N_c\,\vert\,d)g(d)},
{% endraw %}{% raw %}\qquad(28)\]</span></span> where <span class="math inline">\(f(N_c)\)</span> is cancelled. The only stipulation on the possible value of the odds is that it must be a positive value. As we would like to equally weigh odds less than one as those of several hundred or thousand, it is more convenient to compute the log-odds, given as <span id="eq:log_odds"><span class="math display">\[
{% endraw %}{% raw %}\log \mathcal{O}(s\,\vert\,N_c)= \log {g(s) \over g(d)} + \log {f(N_c \,\vert\, s )\over f(N_c\,\vert\, d)}.
{% endraw %}{% raw %}\qquad(29)\]</span></span> Computing the log-transform reveals two interesting quantities. The first term is the ratio of the priors and tells us the <em>a priori</em> knowledge of the odds of survival irrespective of the number of channels. As we have no reason to think that survival is more likely than death, this ratio goes to unity. The second term is the log likelihood ratio and tells us how likely we are to observe a given channel copy number <span class="math inline">\(N_c\)</span> given the cell survives relative to when it dies.</p>
{% endraw %}{% raw %}<p>    For each channel copy number, we can evaluate Eq. 29 to measure the log-odds of survival. If we start with zero channels per cell, we can write the log-odds of survival as <span id="eq:nc_0"><span class="math display">\[
{% endraw %}{% raw %}\log \mathcal{O}(s\,\vert\,N_c=0) = \log {g(s) \over g(d)} + \log {f(N_c=0\,\vert\, s) \over f(N_c=0\,\vert\, d)}.
{% endraw %}{% raw %}\qquad(30)\]</span></span> For a channel copy number of one, the odds of survival is <span id="eq:nc_1"><span class="math display">\[
{% endraw %}{% raw %}\log \mathcal{O}(s\,\vert\,N_c=1) = \log{g(s) \over g(d)} + \log{f(N_c=1\,\vert\, s) \over f(N_c=1\,\vert\, d)}.
{% endraw %}{% raw %}\qquad(31)\]</span></span> In both Eq. 30 and Eq. 31, the log of our <em>a priori</em> knowledge of survival versus death remains. The only factor that is changing is log likelihood ratio. We can be more general in our language and say that the log-odds of survival is increased by the difference in the log-odds conveyed by addition of a single channel. We can rewrite the log likelihood ratio in a more general form as <span id="eq:generalized_LLR"><span class="math display">\[
{% endraw %}{% raw %}\log {f(N_c\, \vert\, s) \over f(N_c\,\vert\, d)} = \log{f(N_c = 0\,\vert\,s) \over f(N_c=0\,\vert\, d)} + N_c \left[\log{f(N_c=1 \,\vert\,s) \over f(N_c=1\,\vert\, d)} - \log{f(N_c=0\,\vert\, s) \over f(N_c=0\,\vert\, d)}\right],
{% endraw %}{% raw %}\qquad(32)\]</span></span> where we are now only considering the case in which <span class="math inline">\(N_c \in [0, 1]\)</span>. The bracketed term in Eq. 32 is the log of the odds of survival given a single channel relative to the odds of survival given no channels. Mathematically, this odds-ratio can be expressed as <span id="eq:odds_ratio"><span class="math display">\[
{% endraw %}{% raw %}\log\mathcal{OR}_{N_c}(s) = \log{{f(N_c=1\,\vert\,s)g(s)\over f(N_c=1\,\vert\,d)g(d)}\over {f(N_c=0\,\vert\,s)g(s)\over f(N_c=0\,\vert\,d)g(d)}} = \log{f(N_c=1\,\vert\,s) \over f(N_c=1\,\vert\,d)} - \log{f(N_c=0\,\vert\,s)\over f(N_c=0\,\vert\,d)} .
{% endraw %}{% raw %}\qquad(33)\]</span></span> Eq. 33 is mathematically equivalent to the bracketed term shown in Eq. 32.</p>
{% endraw %}{% raw %}<p>    We can now begin to staple these pieces together to arrive at an expression for the log odds of survival. Combining Eq. 32 with Eq. 29 yields <span id="eq:combined_v1"><span class="math display">\[
{% endraw %}{% raw %}\log \mathcal{O}(s\,\vert\,N_c) = \log{g(s) \over g(d)} + \log {f(N_c=0\,\vert\, s) \over f(N_c=0\,\vert\, d)} + N_c\left[{f(N_c=1\,\vert\,s)\over f(N_c=1\,\vert\,d)} - \log{f(N_c=0\,\vert\,s) \over f(N_c=0\,\vert\,d)}\right].
{% endraw %}{% raw %}\qquad(34)\]</span></span> Using our knowledge that the bracketed term is the log odds-ratio and the first two times represents the log-odds of survival with <span class="math inline">\(N_c=0\)</span>, we conclude with <span id="eq:bayesian_logit"><span class="math display">\[
{% endraw %}{% raw %}\log\mathcal{O}(s\,\vert\,N_c) = \log \mathcal{O}(s\,\vert\,N_c=0) + N_c \log \mathcal{OR}_{N_c}(s).
{% endraw %}{% raw %}\qquad(35)\]</span></span> This result can be directly compared to Eq. 1 presented in the main text, <span id="eq:"><span class="math display">\[
{% endraw %}{% raw %}\log {p_s \over 1 - p_s} = \beta_0 + \beta_1 N_c,
{% endraw %}{% raw %}\qquad(36)\]</span></span> which allows for an interpretation of the seemingly arbitrary coefficients <span class="math inline">\(\beta_0\)</span> and <span class="math inline">\(\beta_1\)</span>. The intercept term, <span class="math inline">\(\beta_0\)</span>, captures the log-odds of survival with no MscL channels. The slope, <span class="math inline">\(\beta_1\)</span>, describes the log odds-ratio of survival which a single channel relative to the odds of survival with no channels at all. While we have examined this considering only two possible channel copy numbers (<span class="math inline">\(1\)</span> and <span class="math inline">\(0\)</span>), the relationship between them is linear. We can therefore generalize this for any MscL copy number as the increase in the log-odds of survival is constant for addition of a single channel.</p>
{% endraw %}{% raw %}<h3 id="other-properties-as-predictor-variables">Other properties as predictor variables</h3>
{% endraw %}{% raw %}<p>     The previous two sections discuss in detail the logic and practice behind the application of logistic regression to cell survival data using only the effective channel copy number as the predictor of survival. However, there are a variety of properties that could rightly be used as predictor variables, such as cell area and shock rate. As is stipulated in our standard candle calibration, there should be no correlation between survival and cell area. Fig. 7A and B show the logistic regression performed on the cell area. We see for both slow and fast shock groups,there is little change in survival probability with changing cell area and the wide credible regions allow for both positive and negative correlation between survival and area. The appearance of a bottle neck in the notably wide credible regions is a result of a large fraction of the measurements being tightly distributed about a mean value. Fig. 7C shows the predicted survival probability as a function of the the shock rate. There is a slight decrease in survivability as a function of increasing shock rate, however the width of the credible region allows for slightly positive or slightly negative correlation. While we have presented logistic regression in this section as a one-dimensional method, Eq. 19 can be generalized to <span class="math inline">\(n\)</span> predictor variables <span class="math inline">\(x\)</span> as <span id="eq:generalized_logit"><span class="math display">\[
{% endraw %}{% raw %}\log {p_s \over 1 - p_s} = \beta_0 + \sum\limits_{i}^n \beta_ix_i.
{% endraw %}{% raw %}\qquad(37)\]</span></span> Using this generalization, we can use both shock rate and the effective channel copy number as predictor variables. The resulting two-dimensional surface of survival probability is shown in Fig. 7D. As is suggested by Fig. 7C, the magnitude of change in survivability as the shock rate is increased is smaller than that along increasing channel copy number, supporting our conclusion that for MscL alone, the copy number is the most important variable in determining survival.</p>
{% endraw %}{% raw %}<div class="figure">
{% endraw %}{% raw %}<img src="assets/figSX_alternative_predictors.png" alt="Figure 7: Survival probability estimation using alternative predictor variables. (A) Estimated survival probability as a function of cell area for the slow shock group. (B) Estimated survival probability as a function of cell area for the fast shock group. (C) Estimated survival probability as a function shock rate. Black points at top and bottom of plots represent single-cell measurements of cells who survived and perished, respectively. Shaded regions in (A) - (C) represent the 95% credible region. (D) Surface of estimated survival probability using both shock rate and effective channel number as predictor variables. Black points at left and right of plot represent single-cell measurements of cells which survived and died, respectively, sorted by shock rate. Points at top and bottom of plot represent survival and death sorted by their effective channel copy number. Labeled contours correspond to the survival probability." id="fig:alternative_predictor_variables" />
{% endraw %}{% raw %}<p class="caption">Figure 7: Survival probability estimation using alternative predictor variables. (A) Estimated survival probability as a function of cell area for the slow shock group. (B) Estimated survival probability as a function of cell area for the fast shock group. (C) Estimated survival probability as a function shock rate. Black points at top and bottom of plots represent single-cell measurements of cells who survived and perished, respectively. Shaded regions in (A) - (C) represent the 95% credible region. (D) Surface of estimated survival probability using both shock rate and effective channel number as predictor variables. Black points at left and right of plot represent single-cell measurements of cells which survived and died, respectively, sorted by shock rate. Points at top and bottom of plot represent survival and death sorted by their effective channel copy number. Labeled contours correspond to the survival probability.</p>
{% endraw %}{% raw %}</div>
{% endraw %}{% raw %}<table style="width:100%;">
{% endraw %}{% raw %}<caption><em>Escherichia coli</em> strains used in this work.</caption>
{% endraw %}{% raw %}<colgroup>
{% endraw %}{% raw %}<col width="17%" />
{% endraw %}{% raw %}<col width="68%" />
{% endraw %}{% raw %}<col width="13%" />
{% endraw %}{% raw %}</colgroup>
{% endraw %}{% raw %}<thead>
{% endraw %}{% raw %}<tr class="header">
{% endraw %}{% raw %}<th>Strain name</th>
{% endraw %}{% raw %}<th>Genotype</th>
{% endraw %}{% raw %}<th>Reference</th>
{% endraw %}{% raw %}</tr>
{% endraw %}{% raw %}</thead>
{% endraw %}{% raw %}<tbody>
{% endraw %}{% raw %}<tr class="odd">
{% endraw %}{% raw %}<td>MJF641</td>
{% endraw %}{% raw %}<td>Frag1, <em><span class="math inline">\(\Delta\)</span>mscL::cm, <span class="math inline">\(\Delta\)</span>mscS, <span class="math inline">\(\Delta\)</span>mscK::kan, <span class="math inline">\(\Delta\)</span>ybdG::apr, <span class="math inline">\(\Delta\)</span>ynaI, <span class="math inline">\(\Delta\)</span>yjeP, <span class="math inline">\(\Delta\)</span>ybiO, ycjM::Tn10</em></td>
{% endraw %}{% raw %}<td><span class="citation">(11)</span></td>
{% endraw %}{% raw %}</tr>
{% endraw %}{% raw %}<tr class="even">
{% endraw %}{% raw %}<td>MLG910</td>
{% endraw %}{% raw %}<td>MG1655, <em><span class="math inline">\(\Delta\)</span>mscL ::<span class="math inline">\(\phi\)</span>mscL-sfGFP, <span class="math inline">\(\Delta\)</span>galK::kan, <span class="math inline">\(\Delta\)</span>lacI, <span class="math inline">\(\Delta\)</span>lacZY A</em></td>
{% endraw %}{% raw %}<td><span class="citation">(1)</span></td>
{% endraw %}{% raw %}</tr>
{% endraw %}{% raw %}<tr class="odd">
{% endraw %}{% raw %}<td>D6LG-Tn10</td>
{% endraw %}{% raw %}<td>Frag1, <em><span class="math inline">\(\Delta\)</span>mscL ::<span class="math inline">\(\phi\)</span>mscL-sfGFP, <span class="math inline">\(\Delta\)</span>mscS, <span class="math inline">\(\Delta\)</span>mscK::kan, <span class="math inline">\(\Delta\)</span>ybdG::apr, <span class="math inline">\(\Delta\)</span>ynaI, <span class="math inline">\(\Delta\)</span>yjeP, <span class="math inline">\(\Delta\)</span>ybiO, ycjM::Tn10</em></td>
{% endraw %}{% raw %}<td>This work</td>
{% endraw %}{% raw %}</tr>
{% endraw %}{% raw %}<tr class="even">
{% endraw %}{% raw %}<td>D6LG (SD0)</td>
{% endraw %}{% raw %}<td>Frag1, <em><span class="math inline">\(\Delta\)</span>mscL ::<span class="math inline">\(\phi\)</span>mscL-sfGFP, <span class="math inline">\(\Delta\)</span>mscS, <span class="math inline">\(\Delta\)</span>mscK::kan, <span class="math inline">\(\Delta\)</span>ybdG::apr, <span class="math inline">\(\Delta\)</span>ynaI, <span class="math inline">\(\Delta\)</span>yjeP, <span class="math inline">\(\Delta\)</span>ybiO</em></td>
{% endraw %}{% raw %}<td>This work</td>
{% endraw %}{% raw %}</tr>
{% endraw %}{% raw %}<tr class="odd">
{% endraw %}{% raw %}<td>XTL298</td>
{% endraw %}{% raw %}<td>CC4231<em>, araD:: tetA-sacB-amp</em></td>
{% endraw %}{% raw %}<td><span class="citation">(12)</span></td>
{% endraw %}{% raw %}</tr>
{% endraw %}{% raw %}<tr class="even">
{% endraw %}{% raw %}<td>D6LTetSac</td>
{% endraw %}{% raw %}<td>Frag1, <em>mscL-sfGFP:: tetA-sacB, <span class="math inline">\(\Delta\)</span>mscS, <span class="math inline">\(\Delta\)</span>mscK::kan, <span class="math inline">\(\Delta\)</span>ybdG::apr, <span class="math inline">\(\Delta\)</span>ynaI, <span class="math inline">\(\Delta\)</span>yjeP, <span class="math inline">\(\Delta\)</span>ybiO</em></td>
{% endraw %}{% raw %}<td>This work</td>
{% endraw %}{% raw %}</tr>
{% endraw %}{% raw %}<tr class="odd">
{% endraw %}{% raw %}<td>D6LG (SD1)</td>
{% endraw %}{% raw %}<td>Frag1, <em><span class="math inline">\(\Delta\)</span>mscL ::<span class="math inline">\(\phi\)</span>mscL-sfGFP, <span class="math inline">\(\Delta\)</span>mscS, <span class="math inline">\(\Delta\)</span>mscK::kan, <span class="math inline">\(\Delta\)</span>ybdG::apr, <span class="math inline">\(\Delta\)</span>ynaI, <span class="math inline">\(\Delta\)</span>yjeP, <span class="math inline">\(\Delta\)</span>ybiO</em></td>
{% endraw %}{% raw %}<td>This work</td>
{% endraw %}{% raw %}</tr>
{% endraw %}{% raw %}<tr class="even">
{% endraw %}{% raw %}<td>D6LG (SD2)</td>
{% endraw %}{% raw %}<td>Frag1, <em><span class="math inline">\(\Delta\)</span>mscL ::<span class="math inline">\(\phi\)</span>mscL-sfGFP, <span class="math inline">\(\Delta\)</span>mscS, <span class="math inline">\(\Delta\)</span>mscK::kan, <span class="math inline">\(\Delta\)</span>ybdG::apr, <span class="math inline">\(\Delta\)</span>ynaI, <span class="math inline">\(\Delta\)</span>yjeP, <span class="math inline">\(\Delta\)</span>ybiO</em></td>
{% endraw %}{% raw %}<td>This work</td>
{% endraw %}{% raw %}</tr>
{% endraw %}{% raw %}<tr class="odd">
{% endraw %}{% raw %}<td>D6LG (SD4)</td>
{% endraw %}{% raw %}<td>Frag1, <em><span class="math inline">\(\Delta\)</span>mscL ::<span class="math inline">\(\phi\)</span>mscL-sfGFP, <span class="math inline">\(\Delta\)</span>mscS, <span class="math inline">\(\Delta\)</span>mscK::kan, <span class="math inline">\(\Delta\)</span>ybdG::apr, <span class="math inline">\(\Delta\)</span>ynaI, <span class="math inline">\(\Delta\)</span>yjeP, <span class="math inline">\(\Delta\)</span>ybiO</em></td>
{% endraw %}{% raw %}<td>This work</td>
{% endraw %}{% raw %}</tr>
{% endraw %}{% raw %}<tr class="even">
{% endraw %}{% raw %}<td>D6LG (SD6)</td>
{% endraw %}{% raw %}<td>Frag1, <em><span class="math inline">\(\Delta\)</span>mscL ::<span class="math inline">\(\phi\)</span>mscL-sfGFP, <span class="math inline">\(\Delta\)</span>mscS, <span class="math inline">\(\Delta\)</span>mscK::kan, <span class="math inline">\(\Delta\)</span>ybdG::apr, <span class="math inline">\(\Delta\)</span>ynaI, <span class="math inline">\(\Delta\)</span>yjeP, <span class="math inline">\(\Delta\)</span>ybiO</em></td>
{% endraw %}{% raw %}<td>This work</td>
{% endraw %}{% raw %}</tr>
{% endraw %}{% raw %}<tr class="odd">
{% endraw %}{% raw %}<td>D6LG (12SD2)</td>
{% endraw %}{% raw %}<td>Frag1, <em><span class="math inline">\(\Delta\)</span>mscL ::<span class="math inline">\(\phi\)</span>mscL-sfGFP, <span class="math inline">\(\Delta\)</span>mscS, <span class="math inline">\(\Delta\)</span>mscK::kan, <span class="math inline">\(\Delta\)</span>ybdG::apr, <span class="math inline">\(\Delta\)</span>ynaI, <span class="math inline">\(\Delta\)</span>yjeP, <span class="math inline">\(\Delta\)</span>ybiO</em></td>
{% endraw %}{% raw %}<td>This work</td>
{% endraw %}{% raw %}</tr>
{% endraw %}{% raw %}<tr class="even">
{% endraw %}{% raw %}<td>D6LG (16SD0)</td>
{% endraw %}{% raw %}<td>Frag1, <em><span class="math inline">\(\Delta\)</span>mscL ::<span class="math inline">\(\phi\)</span>mscL-sfGFP, <span class="math inline">\(\Delta\)</span>mscS, <span class="math inline">\(\Delta\)</span>mscK::kan, <span class="math inline">\(\Delta\)</span>ybdG::apr, <span class="math inline">\(\Delta\)</span>ynaI, <span class="math inline">\(\Delta\)</span>yjeP, <span class="math inline">\(\Delta\)</span>ybiO</em></td>
{% endraw %}{% raw %}<td>This work</td>
{% endraw %}{% raw %}</tr>
{% endraw %}{% raw %}</tbody>
{% endraw %}{% raw %}</table>
{% endraw %}{% raw %}<table>
{% endraw %}{% raw %}<caption>Oligonucleotide sequences used in this work. Bold and italics correspond to Shine-Dalgarno sequence modifications and <code>AT</code> hairpin insertion modifications, respectively. Double bar <code>||</code> indicates a transposon insertion site.</caption>
{% endraw %}{% raw %}<thead>
{% endraw %}{% raw %}<tr class="header">
{% endraw %}{% raw %}<th align="left">Primer Name</th>
{% endraw %}{% raw %}<th align="left">Sequence (5' <span class="math inline">\(\rightarrow\)</span> 3')</th>
{% endraw %}{% raw %}</tr>
{% endraw %}{% raw %}</thead>
{% endraw %}{% raw %}<tbody>
{% endraw %}{% raw %}<tr class="odd">
{% endraw %}{% raw %}<td align="left"><em>Tn10delR</em></td>
{% endraw %}{% raw %}<td align="left"><code>taaagccaacggcatccaggcggacatactcagca||</code></td>
{% endraw %}{% raw %}</tr>
{% endraw %}{% raw %}<tr class="even">
{% endraw %}{% raw %}<td align="left"></td>
{% endraw %}{% raw %}<td align="left"><code>cctttcgcaaggtaacagagtaaaacatccaccat</code></td>
{% endraw %}{% raw %}</tr>
{% endraw %}{% raw %}<tr class="odd">
{% endraw %}{% raw %}<td align="left"><em>MscLSPSac</em></td>
{% endraw %}{% raw %}<td align="left"><code>gaaaatggcttaacatttgttagacttatggttgtcgg</code></td>
{% endraw %}{% raw %}</tr>
{% endraw %}{% raw %}<tr class="even">
{% endraw %}{% raw %}<td align="left"></td>
{% endraw %}{% raw %}<td align="left"><code>cttcat</code><strong><code>agggag</code></strong><code>TCCTAATTTTTGTTGACACTCTATC</code></td>
{% endraw %}{% raw %}</tr>
{% endraw %}{% raw %}<tr class="odd">
{% endraw %}{% raw %}<td align="left"><em>MscLSPSacR</em></td>
{% endraw %}{% raw %}<td align="left"><code>accacgttcccgcgcatcgcaaattcgcgaaat</code></td>
{% endraw %}{% raw %}</tr>
{% endraw %}{% raw %}<tr class="even">
{% endraw %}{% raw %}<td align="left"></td>
{% endraw %}{% raw %}<td align="left"><code>tctttaataatgctcatATCAAAGGGAAAACTGTCCATA</code></td>
{% endraw %}{% raw %}</tr>
{% endraw %}{% raw %}<tr class="odd">
{% endraw %}{% raw %}<td align="left"><em>MscL-SD1R</em></td>
{% endraw %}{% raw %}<td align="left"><code>atcgcaaattcgcgaaattctttaataatgctcat</code></td>
{% endraw %}{% raw %}</tr>
{% endraw %}{% raw %}<tr class="even">
{% endraw %}{% raw %}<td align="left"></td>
{% endraw %}{% raw %}<td align="left"><code>gttatt</code><strong><code>ctcctc</code></strong><code>atgaagccgacaaccataagtctaacaaa</code></td>
{% endraw %}{% raw %}</tr>
{% endraw %}{% raw %}<tr class="odd">
{% endraw %}{% raw %}<td align="left"><em>MscL-SD2R</em></td>
{% endraw %}{% raw %}<td align="left"><code>atcgcaaattcgcgaaattctttaataatgctcat</code><em><code>gttatt</code></em></td>
{% endraw %}{% raw %}</tr>
{% endraw %}{% raw %}<tr class="even">
{% endraw %}{% raw %}<td align="left"></td>
{% endraw %}{% raw %}<td align="left"><strong><code>tcccct</code></strong><code>atgaagccgacaaccataagtctaacaaa</code></td>
{% endraw %}{% raw %}</tr>
{% endraw %}{% raw %}<tr class="odd">
{% endraw %}{% raw %}<td align="left"><em>MscL-SD4R</em></td>
{% endraw %}{% raw %}<td align="left"><code>atcgcaaattcgcgaaattctttaataatgctcat</code></td>
{% endraw %}{% raw %}</tr>
{% endraw %}{% raw %}<tr class="even">
{% endraw %}{% raw %}<td align="left"></td>
{% endraw %}{% raw %}<td align="left"><em><code>gttatt</code></em> <strong><code>cctgct</code></strong><code>atgaagccgacaaccataagtctaacaaa</code></td>
{% endraw %}{% raw %}</tr>
{% endraw %}{% raw %}<tr class="odd">
{% endraw %}{% raw %}<td align="left"><em>MscL-SD6R</em></td>
{% endraw %}{% raw %}<td align="left"><code>atcgcaaattcgcgaaattctttaataatgctcat</code></td>
{% endraw %}{% raw %}</tr>
{% endraw %}{% raw %}<tr class="even">
{% endraw %}{% raw %}<td align="left"></td>
{% endraw %}{% raw %}<td align="left"><em><code>gttatt</code></em> <strong><code>gctcgt</code></strong><code>atgaagccgacaaccataagtctaacaaa</code></td>
{% endraw %}{% raw %}</tr>
{% endraw %}{% raw %}<tr class="odd">
{% endraw %}{% raw %}<td align="left"><em>MscL-12SD2R</em></td>
{% endraw %}{% raw %}<td align="left"><code>atcgcaaattcgcgaaattctttaataatgctcat</code></td>
{% endraw %}{% raw %}</tr>
{% endraw %}{% raw %}<tr class="even">
{% endraw %}{% raw %}<td align="left"></td>
{% endraw %}{% raw %}<td align="left"><em><code>atatatatatat</code></em> <strong><code>tcccct</code></strong><code>atgaagccgacaaccataagtctaacaaa</code></td>
{% endraw %}{% raw %}</tr>
{% endraw %}{% raw %}<tr class="odd">
{% endraw %}{% raw %}<td align="left"><em>MscL-16SD0R</em></td>
{% endraw %}{% raw %}<td align="left"><code>atcgcaaattcgcgaaattctttaataatgctcat</code></td>
{% endraw %}{% raw %}</tr>
{% endraw %}{% raw %}<tr class="even">
{% endraw %}{% raw %}<td align="left"></td>
{% endraw %}{% raw %}<td align="left"><em><code>atatatatatatatat</code></em> <strong><code>ctccct</code></strong><code>atgaagccgacaaccataagtctaacaaa</code></td>
{% endraw %}{% raw %}</tr>
{% endraw %}{% raw %}</tbody>
{% endraw %}{% raw %}</table>
{% endraw %}